{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "6d4597cb",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import torch\n",
    "from transformers import CLIPUserModel\n",
    "import pickle as pkl\n",
    "from transformers import CLIPTokenizerFast\n",
    "import os\n",
    "from tqdm import tqdm\n",
    "import numpy as np\n",
    "import seaborn as sns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "acdc15b0",
   "metadata": {},
   "outputs": [],
   "source": [
    "class RankingDataset(torch.utils.data.Dataset):\n",
    "    def __init__(self, encodings1, user_id, doc_id):\n",
    "        self.encodings1 = encodings1\n",
    "        self.user_id = user_id\n",
    "        self.doc_id = doc_id\n",
    "\n",
    "    def __getitem__(self, idx):\n",
    "        item = {key: torch.tensor(val[idx]) for key, val in self.encodings1.items()}\n",
    "        item['user_id'] = torch.tensor(self.user_id[idx])\n",
    "        item['doc_id'] = torch.tensor(self.doc_id[idx])\n",
    "        return item\n",
    "\n",
    "    def __len__(self):\n",
    "        return len(self.user_id)\n",
    "    \n",
    "    \n",
    "def getRankedPids(model, val_dataset):\n",
    "    model.cuda()\n",
    "    model.eval()\n",
    "    dataloader = torch.utils.data.DataLoader(val_dataset, batch_size=256)\n",
    "\n",
    "    logits, pids = list(), list()\n",
    "    with torch.no_grad():\n",
    "        for b in tqdm(dataloader):\n",
    "            b = {k:v.cuda() for k, v in b.items()}\n",
    "            output = model(**b)\n",
    "\n",
    "            logits.append(output['logits_per_image'].cpu().numpy()[0,:])\n",
    "            pids.append(b['doc_id'].cpu().numpy())\n",
    "            \n",
    "            \n",
    "    logits = list(np.concatenate(logits))\n",
    "    pids = list(np.concatenate(pids))\n",
    "    _, sorted_pids = zip(*sorted(zip(logits, pids),reverse=True))\n",
    "    \n",
    "    return sorted_pids \n",
    "\n",
    "def getTopNArticles(df, sorted_pids,user_id, n=100):\n",
    "    userPids = set(df[df.user_id == user_id].PID)\n",
    "    sample_pids = list()\n",
    "    i = 0\n",
    "    for l in sorted_pids:\n",
    "        if l not in userPids:\n",
    "            sample_pids.append(l)\n",
    "            i += 1\n",
    "        if i == n:\n",
    "            break\n",
    "            \n",
    "    X = df[df.PID.isin(sample_pids)]\n",
    "    X = X.set_index(\"PID\").loc[sample_pids]\n",
    "    return X"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "d55fc88c",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Load train/val data\n",
    "train_dataset = pkl.load(open(\"../book_data/rankingDataset/train_dataset.pkl\", \"rb\"))\n",
    "test_dataset = pkl.load(open(\"../book_data/rankingDataset/test_dataset.pkl\", \"rb\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "024962ef",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Some weights of CLIPUserModel were not initialized from the model checkpoint at openai/clip-vit-base-patch32 and are newly initialized: ['user_embedder.weight', 'user_projection.weight']\n",
      "You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "<All keys matched successfully>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "model = CLIPUserModel.from_pretrained(\"openai/clip-vit-base-patch32\")\n",
    "\n",
    "#model.load_state_dict(torch.load(\"../results/checkpoint-9500/pytorch_model.bin\"))\n",
    "model.load_state_dict(torch.load(\"../results/checkpoint-22000/pytorch_model.bin\"))\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "ccda7dbd",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Play around"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "baa0db12",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Load DFs\n",
    "df = pd.read_csv(\"../clipData/whole.csv\")\n",
    "train = pd.read_csv(\"../clipData/train.csv\")\n",
    "test = pd.read_csv(\"../clipData/test.csv\")\n",
    "\n",
    "#Load Mappers\n",
    "uid2i = pkl.load(open(\"../clipData/uid2i.pkl\", \"rb\"))\n",
    "i2uid = pkl.load(open(\"../clipData/i2uid.pkl\", \"rb\"))\n",
    "\n",
    "uid2name = pkl.load(open(\"../clipData/uid2name.pkl\", \"rb\"))\n",
    "name2uid = pkl.load(open(\"../clipData/name2uid.pkl\", \"rb\"))\n",
    "\n",
    "#Remove duplicates\n",
    "df_no_dup = df.drop_duplicates(subset =\"PID\",\n",
    "                     keep = False, inplace = False)\n",
    "\n",
    "#Fix data\n",
    "tokenizer = CLIPTokenizerFast.from_pretrained(\"openai/clip-vit-base-patch32\")\n",
    "pids, encodings = list(df_no_dup.PID), tokenizer(list(df_no_dup.Abstract), truncation=True, padding=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "f1c53060",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Find userid from name"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "72fbb43c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>user_id</th>\n",
       "      <th>name</th>\n",
       "      <th>AllName</th>\n",
       "      <th>NameList</th>\n",
       "      <th>UidList</th>\n",
       "      <th>Title</th>\n",
       "      <th>Name</th>\n",
       "      <th>Journal</th>\n",
       "      <th>Year</th>\n",
       "      <th>...</th>\n",
       "      <th>Keywords</th>\n",
       "      <th>Categories</th>\n",
       "      <th>ResearchSubjects</th>\n",
       "      <th>Abstract</th>\n",
       "      <th>FullTextLink</th>\n",
       "      <th>PublicationType</th>\n",
       "      <th>ContentType</th>\n",
       "      <th>PID</th>\n",
       "      <th>JSON</th>\n",
       "      <th>i</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>2709</th>\n",
       "      <td>3042</td>\n",
       "      <td>u1gqsept</td>\n",
       "      <td>Käll, Lukas</td>\n",
       "      <td>['McIlwain, Sean', 'Tamura, Kaipo', 'Kertesz-F...</td>\n",
       "      <td>['Käll, Lukas']</td>\n",
       "      <td>['u1gqsept']</td>\n",
       "      <td>Crux : Rapid Open Source Protein Tandem Mass S...</td>\n",
       "      <td>McIlwain, Sean;Tamura, Kaipo;Kertesz-Farkas, A...</td>\n",
       "      <td>Journal of Proteome Research</td>\n",
       "      <td>2014.0</td>\n",
       "      <td>...</td>\n",
       "      <td>['Peptide Identification', 'Proteomics Data', ...</td>\n",
       "      <td>['Bioinformatik (beräkningsbiologi) (10203)']</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Efficiently and accurately analyzing big prote...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Artikel i tidskrift</td>\n",
       "      <td>Refereegranskat</td>\n",
       "      <td>761918</td>\n",
       "      <td>{\"AllName\":[\"McIlwain, Sean\",\"Tamura, Kaipo\",\"...</td>\n",
       "      <td>1569</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2739</th>\n",
       "      <td>3074</td>\n",
       "      <td>u1gqsept</td>\n",
       "      <td>Käll, Lukas</td>\n",
       "      <td>['Ashwood, C.', 'Bittremieux, W.', 'Deutsch, E...</td>\n",
       "      <td>['Käll, Lukas']</td>\n",
       "      <td>['u1gqsept']</td>\n",
       "      <td>Proceedings of the EuBIC-MS 2020 Developers’ M...</td>\n",
       "      <td>Ashwood, C.;Bittremieux, W.;Deutsch, E. W.;Don...</td>\n",
       "      <td>Etudes Lawrenciennes</td>\n",
       "      <td>2020.0</td>\n",
       "      <td>...</td>\n",
       "      <td>['Benchmark development', 'Bioinformatics', 'C...</td>\n",
       "      <td>['Analytisk kemi (10401)', 'Bioinformatik och ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>The 2020 European Bioinformatics Community for...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Artikel i tidskrift</td>\n",
       "      <td>Refereegranskat</td>\n",
       "      <td>1591617</td>\n",
       "      <td>{\"AllName\":[\"Ashwood, C.\",\"Bittremieux, W.\",\"D...</td>\n",
       "      <td>1569</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows × 21 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      Unnamed: 0   user_id         name  \\\n",
       "2709        3042  u1gqsept  Käll, Lukas   \n",
       "2739        3074  u1gqsept  Käll, Lukas   \n",
       "\n",
       "                                                AllName         NameList  \\\n",
       "2709  ['McIlwain, Sean', 'Tamura, Kaipo', 'Kertesz-F...  ['Käll, Lukas']   \n",
       "2739  ['Ashwood, C.', 'Bittremieux, W.', 'Deutsch, E...  ['Käll, Lukas']   \n",
       "\n",
       "           UidList                                              Title  \\\n",
       "2709  ['u1gqsept']  Crux : Rapid Open Source Protein Tandem Mass S...   \n",
       "2739  ['u1gqsept']  Proceedings of the EuBIC-MS 2020 Developers’ M...   \n",
       "\n",
       "                                                   Name  \\\n",
       "2709  McIlwain, Sean;Tamura, Kaipo;Kertesz-Farkas, A...   \n",
       "2739  Ashwood, C.;Bittremieux, W.;Deutsch, E. W.;Don...   \n",
       "\n",
       "                           Journal    Year  ...  \\\n",
       "2709  Journal of Proteome Research  2014.0  ...   \n",
       "2739          Etudes Lawrenciennes  2020.0  ...   \n",
       "\n",
       "                                               Keywords  \\\n",
       "2709  ['Peptide Identification', 'Proteomics Data', ...   \n",
       "2739  ['Benchmark development', 'Bioinformatics', 'C...   \n",
       "\n",
       "                                             Categories ResearchSubjects  \\\n",
       "2709      ['Bioinformatik (beräkningsbiologi) (10203)']              NaN   \n",
       "2739  ['Analytisk kemi (10401)', 'Bioinformatik och ...              NaN   \n",
       "\n",
       "                                               Abstract FullTextLink  \\\n",
       "2709  Efficiently and accurately analyzing big prote...          NaN   \n",
       "2739  The 2020 European Bioinformatics Community for...          NaN   \n",
       "\n",
       "          PublicationType      ContentType      PID  \\\n",
       "2709  Artikel i tidskrift  Refereegranskat   761918   \n",
       "2739  Artikel i tidskrift  Refereegranskat  1591617   \n",
       "\n",
       "                                                   JSON     i  \n",
       "2709  {\"AllName\":[\"McIlwain, Sean\",\"Tamura, Kaipo\",\"...  1569  \n",
       "2739  {\"AllName\":[\"Ashwood, C.\",\"Bittremieux, W.\",\"D...  1569  \n",
       "\n",
       "[2 rows x 21 columns]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df[df.name.str.contains(\"Käll, L\")].head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "0ad11e48",
   "metadata": {},
   "outputs": [],
   "source": [
    "#let's use Johan Boye\n",
    "user_id = \"u1gqsept\"\n",
    "uids = [uid2i[user_id]] * len(pids)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "527b070d",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Create dataset\n",
    "val_dataset =  RankingDataset(encodings, uids, pids)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "4070de7f",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 136/136 [00:13<00:00, 10.13it/s]\n"
     ]
    }
   ],
   "source": [
    "sorted_pids = getRankedPids(model, val_dataset)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "5c1fb3e2",
   "metadata": {},
   "outputs": [],
   "source": [
    "X = getTopNArticles(df_no_dup, sorted_pids, user_id, 50)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "e7b0101f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Statistical and machine learning methods to analyze large-scale mass spectrometry data \n",
      "\n",
      "['The, Matthew'] \n",
      "\n",
      "Modern biology is faced with vast amounts of data that contain valuable information yet to be extracted. Proteomics, the study of proteins, has repositories with thousands of mass spectrometry experiments. These data gold mines could further our knowledge of proteins as the main actors in cell processes and signaling. Here, we explore methods to extract more information from this data using statistical and machine learning methods.First, we present advances for studies that aggregate hundreds of runs. We introduce MaRaCluster, which clusters mass spectra for large-scale datasets using statistical methods to assess similarity of spectra. It identified up to 40% more peptides than the state-of-the-art method, MS-Cluster. Further, we accommodated large-scale data analysis in Percolator, a popular post-processing tool for mass spectrometry data. This reduced the runtime for a draft human proteome study from a full day to 10 minutes.Second, we clarify and promote the contentious topic of protein false discovery rates (FDRs). Often, studies report lists of proteins but fail to report protein FDRs. We provide a framework to systematically discuss protein FDRs and take away hesitance. We also added protein FDRs to Percolator, opting for the best-peptide approach which proved superior in a benchmark of scalable protein inference methods.Third, we tackle the low sensitivity of protein quantification methods. Current methods lack proper control of error sources and propagation. To remedy this, we developed Triqler, which controls the protein quantification FDR through a Bayesian framework. We also introduce MaRaQuant, which proposes a quantification-first approach that applies clustering prior to identification. This reduced the number of spectra to be searched and allowed us to spot unidentified analytes of interest. Combining these tools outperformed the state-of-the-art method, MaxQuant/Perseus, and found enriched functional terms for datasets that had none before. \n",
      "\n",
      "------- \n",
      "\n",
      "Mapping the human proteome using bioinformatic methods \n",
      "\n",
      "['Fagerberg, Linn'] \n",
      "\n",
      "The fundamental goal of proteomics is to gain an understanding of the expression and function of the proteome on the level of individual proteins, on the level of defined cell types and on the level of the entire organism. In this thesis, the human proteome is explored using membrane protein topology prediction methods to define the human membrane proteome and by global protein expression profiling, which relies on a complex study of the location and expression levels of proteins in tissues and cells.A whole-proteome analysis was performed based on the predicted protein-coding genes of humans using a selection of membrane protein topology prediction methods. The study used a majority decision-based method, which estimated that approximately 26% of the human genes encode for a membrane protein. The prediction results are displayed in a visualization tool to facilitate the selection of antigens to be used for antibody generation.Global protein expression profiles in a large number of cells and tissues in the human body were analyzed for more than 4000 protein targets, based on data from the antibody-based immunohistochemistry and immunofluorescence methods within the framework of the Human Protein Atlas project. The results revealed few cell-type specific proteins and a high fraction of human proteins expressed in most cells, suggesting that cell and tissue specificity is attained by a fine-tuned regulation of protein levels. The expression profiles were also used to analyze the relationship between 45 cell lines by hierarchical clustering and principal component analysis. The global protein expression patterns overall reflected the tumor origin of the cells, and also allowed for identification of proteins of importance for distinguishing different categories of cell lines, as defined by phenotype of progenitor cell. In addition, the protein distribution in 16 subcellular compartments in three of the human cell lines was mapped. A large fraction of proteins were localized in two or more compartments and, in line with previous results, a majority of proteins were detected in all three cell lines.Finally, mass spectrometry-based protein expression levels were compared to RNA-seq-based transcript expression levels in three cell lines. Highly ubiquitous mRNA expression was found and the changes of expression levels between the cell lines showed high correlations between proteins and transcripts. Large general differences in abundance of proteins from various functional classes were observed. A comparison between categories based on expression levels revealed that, in general, genes with varying expression levels between the cell lines or only expressed in one cell line were highly enriched for cell-surface proteins.These studies show a path for a systematic analysis to characterize the proteome in human cells, tissues and organs. \n",
      "\n",
      "------- \n",
      "\n",
      "Advancing Cell Biology Through Proteomics in Space and Time (PROSPECTS) \n",
      "\n",
      "['Lamond, Angus', 'Uhlén, Mathias', 'Horning, Stevan', 'Makarov, Alexander', 'Robinson, Carol', 'Serrano, Luis', 'Hartl, F.', 'Baumeister, Wolfgang', 'Werenskiold, Anne', 'Andersen, Jens', 'Vorm, Ole', 'Linial, Michal', 'Aebersold, Ruedi', 'Mann, Matthias'] \n",
      "\n",
      "The term \"proteomics\" encompasses the large-scale detection and analysis of proteins and their post-translational modifications. Driven by major improvements in mass spectrometric instrumentation, methodology, and data analysis, the proteomics field has burgeoned in recent years. It now provides a range of sensitive and quantitative approaches for measuring protein structures and dynamics that promise to revolutionize our understanding of cell biology and molecular mechanisms in both human cells and model organisms. The Proteomics Specification in Time and Space (PROSPECTS) Network is a unique EU-funded project that brings together leading European research groups, spanning from instrumentation to biomedicine, in a collaborative five year initiative to develop new methods and applications for the functional analysis of cellular proteins. This special issue of Molecular and Cellular Proteomics presents 16 research papers reporting major recent progress by the PROSPECTS groups, including improvements to the resolution and sensitivity of the Orbitrap family of mass spectrometers, systematic detection of proteins using highly characterized antibody collections, and new methods for absolute as well as relative quantification of protein levels. Manuscripts in this issue exemplify approaches for performing quantitative measurements of cell proteomes and for studying their dynamic responses to perturbation, both during normal cellular responses and in disease mechanisms. Here we present a perspective on how the proteomics field is moving beyond simply identifying proteins with high sensitivity toward providing a powerful and versatile set of assay systems for characterizing proteome dynamics and thereby creating a new \"third generation\"proteomics strategy that offers an indispensible tool for cell biology and molecular medicine. \n",
      "\n",
      "------- \n",
      "\n",
      "Statistical and machine learning methods to analyze large-scale mass spectrometry data \n",
      "\n",
      "['The, Matthew'] \n",
      "\n",
      "As in many other fields, biology is faced with enormous amounts ofdata that contains valuable information that is yet to be extracted. The field of proteomics, the study of proteins, has the luxury of having large repositories containing data from tandem mass-spectrometry experiments, readily accessible for everyone who is interested. At the same time, there is still a lot to discover about proteins as the main actors in cell processes and cell signaling.In this thesis, we explore several methods to extract more information from the available data using methods from statistics and machine learning. In particular, we introduce MaRaCluster, a new method for clustering mass spectra on large-scale datasets. This method uses statistical methods to assess similarity between mass spectra, followed by the conservative complete-linkage clustering algorithm.The combination of these two resulted in up to 40% more peptide identifications on its consensus spectra compared to the state of the art method.Second, we attempt to clarify and promote protein-level false discovery rates (FDRs). Frequently, studies fail to report protein-level FDRs even though the proteins are actually the entities of interest. We provided a framework in which to discuss protein-level FDRs in a systematic manner to open up the discussion and take away potential hesitance. We also benchmarked some scalable protein inference methods and included the best one in the Percolator package. Furthermore, we added functionality to the Percolator package to accommodate the analysis of studies in which many runs are aggregated. This reduced the run time for a recent study regarding a draft human proteome from almost a full day to just 10 minutes on a commodity computer, resulting in a list of proteins together with their corresponding protein-level FDRs. \n",
      "\n",
      "------- \n",
      "\n",
      "Diversion of the long-chain acyl-ACP pool in Synechocystis to fatty alcohols through CRISPRi repression of the essential phosphate acyltransferase PlsX \n",
      "\n",
      "['Kaczmarzyk, Danuta', 'Cengic, Ivana', 'Yao, Lun', 'Hudson, Elton'] \n",
      "\n",
      "Fatty alcohol production in Synechocystis sp. PCC 6803 was achieved through heterologous expression of the fatty acyl-CoA/ACP reductase Maqu2220 from the bacteria Marinobacter aquaeolei VT8 and the fatty acyl-ACP reductase DPW from the rice Oryza sativa. These platform strains became models for testing multiplex CRISPR-interference (CRISPRi) metabolic engineering strategies to both improve fatty alcohol production and to study membrane homeostasis. CRISPRi allowed partial repression of up to six genes simultaneously, each encoding enzymes of acyl-ACP-consuming pathways. We identified the essential phosphate acyltransferase enzyme PlsX (slr1510) as a key node in C18 fatty acyl-ACP consumption, repression of slr1510 increased octadecanol productivity threefold over the base strain and gave the highest specific titers reported for this host, 10.3 mg g−1 DCW. PlsX catalyzes the first committed step of phosphatidic acid synthesis, and has not been characterized in Synechocystis previously. We found that accumulation of fatty alcohols impaired growth, altered the membrane composition, and caused a build-up of reactive oxygen species. \n",
      "\n",
      "------- \n",
      "\n",
      "National Cancer Institute Think-Tank Meeting Report on Proteomic Cartography and Biomarkers at the Single-Cell Level : Interrogation of Premalignant Lesions \n",
      "\n",
      "['Kagan, Jacob', 'Moritz, Robert', 'Mazurchuk, Richard', 'Lee, Je', 'Kharchenko, Peter', 'Rozenblatt-Rosen, Orit', 'Ruppin, Eytan', 'Edfors, Fredrik', 'Ginty, Fiona', 'Goltsev, Yury', 'Wells, James', 'LaCava, John', 'Riesterer, Jessica', 'Germain, Ronald', 'Shi, Tujin', 'Chee, Mark', 'Budnik, Bogdan', 'Yates, John', 'Chait, Brian', 'Moffitt, Jeffery', 'Harvard Univ, Med', 'Smith, Richard', 'Srivastava, Sudhir'] \n",
      "\n",
      "A Think-Tank Meeting was convened by the National Cancer Institute (NCI) to solicit experts' opinion on the development and application of multiomic single-cell analyses, and especially single-cell proteomics, to improve the development of a new generation of biomarkers for cancer risk, early detection, diagnosis, and prognosis as well as to discuss the discovery of new targets for prevention and therapy. It is anticipated that such markers and targets will be based on cellular, subcellular, molecular, and functional aberrations within the lesion and within individual cells. Single-cell proteomic data will be essential for the establishment of new tools with searchable and scalable features that indude spatial and temporal cartographies of premalignant and malignant lesions. Challenges and potential solutions that were discussed included (i) The best way/s to analyze single-cells from fresh and preserved tissue; (ii) Detection and analysis of secreted molecules and from single cells, especially from a tissue slice; (iii) Detection of new, previously undocumented cell type/s in the premalignant and early stage cancer tissue microenvironment; (iv) Multiomic integration of data to support and inform proteomic measurements; (v) Subcellular organelles-identifying abnormal structure, function, distribution, and location within individual premalignant and malignant cells; (vi) How to improve the dynamic range of single-cell proteomic measurements for discovery of differentially expressed proteins and their post-translational modifications (PTM); (vii) The depth of coverage measured concurrently using single-cell techniques; (viii) Quantitation - absolute or semiquantitative? (ix) Single methodology or multiplexed combinations? (x) Application of analytical methods for identification of biologically significant subsets; (xi) Data visualization of N-dimensional data sets; (xii) How to construct intercellular signaling networks in individual cells within premalignant tumor microenvironments (TME); (xiii) Associations between intrinsic cellular processes and extrinsic stimuli; (xiv) How to predict cellular responses to stress-inducing stimuli; (xv) Identification of new markers for prediction of progression from precursor, benign, and localized lesions to invasive cancer, based on spatial and temporal changes within individual cells; (xvi) Identification of new targets for immunoprevention or immunotherapy-identification of neoantigens and surfactome of individual cells within a lesion. \n",
      "\n",
      "------- \n",
      "\n",
      "Systemic Hypoxia Increases the Expression of DPP4 in Preadipocytes of Healthy Human Participants \n",
      "\n",
      "['Chowdhury, Helena', 'Velebit, Jelena', 'Mekjavic, Igor', 'Eiken, Ola', 'Kreft, Marko', 'Zorec, Robert'] \n",
      "\n",
      "Dipeptidyl peptidase 4 (DPP4) is a transmembrane glycoprotein involved in protein degradation. Due to its action on incretins, which increase insulin secretion, DPP4 is considered a therapeutic target for type 2 diabetes. Here we have studied the role of single and combined effects of hypoxia and inactivity on the expression of DPP4 in human adipose tissue of 12 adult normal-weight males. Fat biopsies were obtained at baseline and after each of three experimental campaigns. The results revealed that in isolated human preadipocytes the expression of DPP4 was significantly increased by exposure of participants to hypoxia. Physical inactivity per se had no apparent effect on the DPP4 expression. It is concluded that DPP4 may be a marker to monitor indirectly tissue hypoxia, as occurs in obese subjects. \n",
      "\n",
      "------- \n",
      "\n",
      "Droplet microfluidics for high throughput biological analysis \n",
      "\n",
      "['Jönsson, Håkan'] \n",
      "\n",
      "Many areas of biological research increasingly perform large-scale analyses.  In genomics the entire gene repertoire of an organism is analyzed.  Proteomics attempts to understand the function and expression patterns of all proteins in a cell or organism.  Cell biologists study large numbers of single cells to understand the heterogeneity of cell populations.  In biotechnology and synthetic biology researchers search for new functional biomolecules in large libraries of biomolecular diversity e.g. for uses in medicine or bioprocessing.  More and more all of these fields employ high throughput methods to achieve the scale of analysis necessary.Miniaturization and parallelization provide routes towards high throughput analysis, which have proven successful for microelectronics as well as for DNA sequencing.  For the analysis of cells and biomolecules, native to an aqueous environment, miniaturization and parallelization hinges on the handling and parallel processing of very small amounts of water.  Droplet microfluidics utilizes stable picoliter (water) droplets contained in inert fluorinated oils as compartments in which to isolate and analyze cells, molecules or reactions.  These droplets can be manipulated, detected and analyzed at rates of thousands per second in microfluidic modules combining top-down microscale fabrication with the self-assembly of droplets of exact size.The studies constituting this thesis involve new droplet based biomolecular and single cell assays, manipulation techniques and device fabrication methods to extend the capabilities of droplet microfluidics for high throughput biological analysis.The first paper in the thesis describes a novel analysis method for studying the low abundant biomarkers present on the surface individual cells at resolutions not available by flow cytometry, the current gold standard of single cell analysis.  The use of a fluorescent optical dye code enabled the analysis of several single cell samples concurrently, improving throughput.Further a deterministic lateral displacement module, providing passive separation of droplets by size in a microfluidic circuit at more than twice higher rates than previously achievable was demonstrated.  Using this module, droplets were separated for cell occupancy based on a cell induced droplet size transformation, which couples a biological property of the droplet contents to a physical property of the droplet.  This effect, which enables passive separation of at high throughput, indicates a potential novel assay format for clone selection.One important feature of droplets for encapsulated single cell analysis is retention of secreted molecules providing a genotype-phenotype link.  With the objective of detecting antibody molecules secreted by hybridoma for selection, Paper III demonstrates the adaption of a homogeneous fluorescence polarization based, “mix-incubate-read”, assay for antibody detection.  In the final paper of the thesis the development of inexpensive and robust optical filters monolithically integrated in the microfluidic chip is reported. These defined filters enable integration of multiple optical filters in a polymer microfluidic device.Overall, droplet microfluidics combines techniques for handling and manipulating millions of discrete biocompatible picoliter compartments per hour with dedicated assays for biomolecule and single cell analysis. The scale of analysis that this enables is certain to impact life science research.  \n",
      "\n",
      "------- \n",
      "\n",
      "Effects of Successive Peak Flow Events on Hyporheic Exchange and Residence Times \n",
      "\n",
      "['Singh, Tanu', 'Gomez-Velez, Jesus', 'Wu, Liwen', 'Wörman, Anders', 'Hannah, David', 'Krause, Stefan'] \n",
      "\n",
      "Hyporheic exchange is a crucial control of the type and rates of streambed biogeochemical processes, including metabolism, respiration, nutrient turnover, and the transformation of pollutants. Previous work has shown that increasing discharge during an individual peak flow event strengthens biogeochemical turnover by enhancing the exchange of water and dissolved solutes. However, due to the nonsteady nature of the exchange process, successive peak flow events do not exhibit proportional variations in residence time and turnover, and in some cases, can reduce the hyporheic zones' biogeochemical potential. Here, we used a process-based model to explore the role of successive peak flow events on the flow and transport characteristics of bedform-induced hyporheic exchange. We conducted a systematic analysis of the impacts of the events' magnitude, duration, and time between peaks in the hyporheic zone's fluxes, penetration, and residence times. The relative contribution of each event to the transport of solutes across the sediment-water interface was inferred from transport simulations of a conservative solute. In addition to temporal variations in the hyporheic flow field, our results demonstrate that the separation between two events determines the temporal evolution of residence time and that event time lags longer than the memory of the system result in successive events that can be treated independently. This study highlights the importance of discharge variability in the dynamics of hyporheic exchange and its potential implications for biogeochemical transformations and fate of contaminants along river corridors. Key Points Dynamic response of hyporheic exchange fluxes, residence times, and breakthrough curves to successive peak flow events is investigated Increased time lag between successive flow peaks cause higher temporal variability in mean residence times of hyporheic water Two events can be treated separately if the occurrence of subsequent event is longer than the memory of the system from the antecedent event. \n",
      "\n",
      "------- \n",
      "\n",
      "Hydrodynamics considerations in cells systems from ocean flow to perfusion cultivation process \n",
      "\n",
      "['Zhan, Caijuan'] \n",
      "\n",
      "Microorganisms and animal cells are grown surrounded by fluid, which is providing them with nutrients and removing their waste products. In nature and industry processes, cells/microbes can be subject to aggressive environments, such as turbulent flow or shear flow. Hydrodynamics force generated in these flows can affect the distribution of cells/microbes and even lead to cell damage. Understanding the mechanism and exploring the effect of hydrodynamic force in these environments could make the prediction of cells’ hydrodynamic response more systematic. In pharmaceutical industry, perfusion process is recognized as an attractive option for biologics production due to its high productivity. However, there are still some challenges and limitations for further process improvement due to lack of information of cell response to hydrodynamic force and nutrients. In both cases, hydrodynamics plays an important role and similar tool can be used to achieve a deeper understanding of these processes. This thesis is mainly aiming to elucidate the influence of hydrodynamic forces on microorganisms or cells in nature and during bioprocesses. In particular, shear stress in a natural environment and in a bioreactor operated in perfusion mode is studied.This work mainly investigates hydrodynamics in nature and bioprocess including three flow cases. The first study investigates the effect of turbulence on marine life by performing direct numerical simulations (DNS) of motile micro-organisms in isotropic homogeneous turbulence. The clustering level of micro-organisms with one preferential swimming direction (e.g. gyrotaxis) is examined. The second study uses Computation Fluid Dynamics (CFD) to simulate the fluid flow inside a Wave bioreactor bag. The phenomenon of mixing, oxygen transfer rate and shear stress in nine different operating conditions of rocking speeds and angles are discussed. In the third study, the cellular response to shear force including growth and metabolism in a cell retention device such as hollow fiber filters during a perfusion process is analyzed. Theoretical calculations and experiment validation is performed to compare two filtration modes, tangential flow filtration (TFF) or alternating tangential flow filtration (ATF). Further optimizations regarding mixing and feeding are performed in a screening scale of in a perfusion system of stirred tank bioreactor with cell separation device.The main findings can be summarized as that spherical gyrotaxis swimmers show significant clustering, whereas prolate swimmers remain more uniformly distributed due to their large sensitivity to the local shear. These results could explain how pure hydrodynamic effects can alter the ecology of micro-organisms for instance by varying shape and their preferential orientation (paper I). The simulations of Wave bioreactors show that the mixing and shear stress increase with the rocking angle but that increasing rocking speeds are not systematically associated with increasing mixing and shear stress. A resonance phenomenon is responsible for the fact that the lowest studied rocking speed generates the highest fluid velocity, mixing and shear stress (paper II). Theoretical velocity profile-based calculations suggested a lower shear stress for ATF by a factor 0.637 compared to TFF. This is experimentally validated by cultures of HEK (human embryonic kidney) 293 cells subjected to shear stress by a perfusion system that affects growth and metabolism using these cell separation devices (paper III). Thanks to optimization of mixing and oxygen transfer in a screening system for perfusion process, very high cell densities above 100 x 106 cells/mL of mammalian cells were achieved (paper IV). \n",
      "\n",
      "------- \n",
      "\n",
      "Bioinformatic Methods in Metagenomics \n",
      "\n",
      "['Alneberg, Johannes'] \n",
      "\n",
      "Microbial organisms are a vital part of our global ecosystem. Yet, our knowledge of them is still lacking. Direct sequencing of microbial communities, i.e. metagenomics, have enabled detailed studies of these microscopic organisms by inspection of their DNA sequences without the need to culture them. Furthermore, the development of modern high- throughput sequencing technologies have made this approach more powerful and cost-effective. Taken together, this has shifted the field of microbiology from previously being centered around microscopy and culturing studies, to largely consist of computational analyses of DNA sequences. One such computational analysis which is the main focus of this thesis, aims at reconstruction of the complete DNA sequence of an organism, i.e. its genome, directly from short metagenomic sequences.This thesis consists of an introduction to the subject followed by five papers. Paper I describes a large metagenomic data resource spanning the Baltic Sea microbial communities. This dataset is complemented with a web-interface allowing researchers to easily extract and visualize detailed information. Paper II introduces a bioinformatic method which is able to reconstruct genomes from metagenomic data. This method, which is termed CONCOCT, is applied on Baltic Sea metagenomics data in Paper III and Paper V. This enabled the reconstruction of a large number of genomes. Analysis of these genomes in Paper III led to the proposal of, and evidence for, a global brackish microbiome. Paper IV presents a comparison between genomes reconstructed from metagenomes with single-cell sequenced genomes. This further validated the technique presented in Paper II as it was found to produce larger and more complete genomes than single-cell sequencing.;Mikrobiella organismer är en vital del av vårt globala ekosystem. Trots detta är vår kunskap om dessa fortfarande begränsad. Sekvensering direkt applicerad på mikrobiella samhällen, så kallad metagenomik, har möjliggjort detaljerade studier av dessa mikroskopiska organismer genom deras DNA-sekvenser. Utvecklingen av modern sekvenseringsteknik har vidare gjort denna strategi både mer kraftfull och mer kostnadseffektiv. Sammantaget har detta förändrat mikrobiologi-fältet, från att ha varit centrerat kring mikroskopi, till att till stor del bero på dataintensiva analyser av DNA-sekvenser. En sådan analys, som är det huvudsakliga fokuset för den här avhandlingen, syftar till att återskapa den kompletta DNA-sekvensen för en organism, dvs. dess genom, direkt från korta metagenom-sekvenser.Den här avhandlingen består av en introduktion till ämnet, följt av fem artiklar. Artikel I beskriver en omfattande databas för metagenomik över Östersjöns mikrobiella samhällen. Till denna databas hör också en webbsida som ger forskare möjlighet att lätt extrahera och visualisera detaljerad information. Artikel II introducerar en bioinformatisk metod som kan återskapa genom från metagenom. Denna metod, som kallas CONCOCT, används för data från Östersjön i artikel III och Artikel V. Detta möjliggjorde återskapandet av ett stort antal genom. Analys av dessa genom presenterad i Artikel III ledde till hypotesen om, och belägg för, ett globalt brackvattenmikrobiom. Artikel IV innehåller en jämförelse mellan genom återskapade från metagenom och individuellt sekvenserade genom. Detta validerade metoden som presenterades i Artikel II ytterligare då denna metod visade sig producera större och mer kompletta genom än sekvensering av individuella celler. \n",
      "\n",
      "------- \n",
      "\n",
      "Integrative annotation of 21,037 human genes validated by full-length cDNA clones \n",
      "\n",
      "['Imanishi, T.', 'Itoh, T.', 'Suzuki, Y.', \"O'Donovan, C.\", 'Fukuchi, S.', 'Koyanagi, K.', 'Barrero, R.', 'Tamura, T.', 'Yamaguchi-Kabata, Y.', 'Tanino, M.', 'Yura, K.', 'Miyazaki, S.', 'Ikeo, K.', 'Homma, K.', 'Kasprzyk, A.', 'Nishikawa, T.', 'Hirakawa, M.', 'Thierry-Mieg, J.', 'Thierry-Mieg, D.', 'Ashurst, J.', 'Jia, L.', 'Nakao, M.', 'Thomas, M.', 'Mulder, N.', 'Karavidopoulou, Y.', 'Jin, L.', 'Kim, S.', 'Yasuda, T.', 'Lenhard, B.', 'Eveno, E.', 'Yamasaki, C.', 'Takeda, J.', 'Gough, C.', 'Hilton, P.', 'Fujii, Y.', 'Sakai, H.', 'Tanaka, S.', 'Amid, C.', 'Bellgard, M.', 'de Fatima Bonaldo, M.', 'Bono, H.', 'Bromberg, S.', 'Brookes, A.', 'Bruford, E.', 'Carninci, P.', 'Chelala, C.', 'Couillault, C.', 'de Souza, S.', 'Debily, M.', 'Devignes, M.', 'Dubchak, I.', 'Endo, T.', 'Estreicher, A.', 'Eyras, E.', 'Fukami-Kobayashi, K.', 'Gopinath, G.', 'Graudens, E.', 'Hahn, Y.', 'Han, M.', 'Han, Z.', 'Hanada, K.', 'Hanaoka, H.', 'Harada, E.', 'Hashimoto, K.', 'Hinz, U.', 'Hirai, M.', 'Hishiki, T.', 'Hopkinson, I.', 'Imbeaud, S.', 'Inoko, H.', 'Kanapin, A.', 'Kaneko, Y.', 'Kasukawa, T.', 'Kelso, J.', 'Kersey, P.', 'Kikuno, R.', 'Kimura, K.', 'Korn, B.', 'Kuryshev, V.', 'Makalowska, I.', 'Makino, T.', 'Mano, S.', 'Mariage-Samson, R.', 'Mashima, J.', 'Matsuda, H.', 'Mewes, H.', 'Minoshima, S.', 'Nagai, K.', 'Nagasaki, H.', 'Nagata, N.', 'Nigam, R.', 'Ogasawara, O.', 'Ohara, O.', 'Ohtsubo, M.', 'Okada, N.', 'Okido, T.', 'Oota, S.', 'Ota, M.', 'Ota, T.', 'Otsuki, T.', 'Piatier-Tonneau, D.', 'Poustka, A.', 'Ren, S.', 'Saitou, N.', 'Sakai, K.', 'Sakamoto, S.', 'Sakate, R.', 'Schupp, I.', 'Servant, F.', 'Sherry, S.', 'Shiba, R.', 'Shimizu, N.', 'Shimoyama, M.', 'Simpson, A.', 'Soares, B.', 'Steward, C.', 'Suwa, M.', 'Suzuki, M.', 'Takahashi, A.', 'Tamiya, G.', 'Tanaka, H.', 'Taylor, T.', 'Terwilliger, J.', 'Unneberg, Per', 'Veeramachaneni, V.', 'Watanabe, S.', 'Wilming, L.', 'Yasuda, N.', 'Hyang-Yoo, S.', 'Stodolsky, M.', 'Makalowski, W.', 'Go, M.', 'Nakai, K.', 'Takagi, T.', 'Kanehisa, M.', 'Sakaki, Y.', 'Quackenbush, J.', 'Okazaki, Y.', 'Hayashizaki, Y.', 'Hide, W.', 'Chakraborty, R.', 'Nishikawa, K.', 'Sugawara, H.', 'Tateno, Y.', 'Chen, Z.', 'Oishi, M.', 'Tonellato, P.', 'Apweiler, R.', 'Okubo, K.', 'Wagner, L.', 'Wiemann, S.', 'Strausberg, R.', 'Isogai, T.', 'Auffray, C.', 'Nomura, N.', 'Gojobori, T.', 'Sugano, S.'] \n",
      "\n",
      "The human genome sequence defines our inherent biological potential; the realization of the biology encoded therein requires knowledge of the function of each gene. Currently, our knowledge in this area is still limited. Several lines of investigation have been used to elucidate the structure and function of the genes in the human genome. Even so, gene prediction remains a difficult task, as the varieties of transcripts of a gene may vary to a great extent. We thus performed an exhaustive integrative characterization of 41,118 full-length cDNAs that capture the gene transcripts as complete functional cassettes, providing an unequivocal report of structural and functional diversity at the gene level. Our international collaboration has validated 21,037 human gene candidates by analysis of high-quality full-length cDNA clones through curation using unified criteria. This led to the identification of 5,155 new gene candidates. It also manifested the most reliable way to control the quality of the cDNA clones. We have developed a human gene database, called the H-Invitational Database (H-InvDB; http://www.h-invitational.jp/). It provides the following: integrative annotation of human genes, description of gene structures, details of novel alternative splicing isoforms, non-protein-coding RNAs, functional domains, subcellular localizations, metabolic pathways, predictions of protein three-dimensional structure, mapping of known single nucleotide polymorphisms (SNPs), identification of polymorphic microsatellite repeats within human genes, and comparative results with mouse full-length cDNAs. The H-InvDB analysis has shown that up to 4% of the human genome sequence (National Center for Biotechnology Information build 34 assembly) may contain misassembled or missing regions. We found that 6.5% of the human gene candidates (1,377 loci) did not have a good protein-coding open reading frame, of which 296 loci are strong candidates for non-protein-coding RNA genes. In addition, among 72,027 uniquely mapped SNPs and insertions/deletions localized within human genes, 13,215 nonsynonymous SNPs, 315 nonsense SNPs, and 452 indels occurred in coding regions. Together with 25 polymorphic microsatellite repeats present in coding regions, they may alter protein structure, causing phenotypic effects or resulting in disease. The H-InvDB platform represents a substantial contribution to resources needed for the exploration of human biology and pathology. \n",
      "\n",
      "------- \n",
      "\n",
      "Recommendations from the 2008 International Summit on Proteomics Data Release and Sharing Policy : The Amsterdam Principles \n",
      "\n",
      "['Rodriguez, Henry', 'Snyder, Mike', 'Uhlén, Mathias', 'Andrews, Phil', 'Beavis, Ronald', 'Borchers, Christoph', 'Chalkley, Robert', 'Cho, Sang', 'Cottingham, Katie', 'Dunn, Michael', 'Dylag, Tomasz', 'Edgar, Ron', 'Hare, Peter', 'Heck, Albert', 'Hirsch, Roland', 'Kennedy, Karen', 'Kolar, Patrik', 'Kraus, Hans-Joachim', 'Mallick, Parag', 'Nesvizhskii, Alexey', 'Ping, Peipei', 'Ponten, Fredrik', 'Yang, Liming', 'Yates, John', 'Stein, Stephen', 'Hermjakob, Henning', 'Kinsinger, Christopher', 'Apweiler, Rolf'] \n",
      "\n",
      "Policies supporting the rapid and open sharing of genomic data have directly fueled the accelerated pace of discovery in large-scale genomics research. The proteomics community is starting to implement analogous policies and infrastructure for making large-scale proteomics data widely available on a precompetitive basis. On August 14, 2008, the National Cancer Institute (NCI) convened the \"International Summit on Proteomics Data Release and Sharing Policy\" in Amsterdam, The Netherlands, to identify and address potential roadblocks to rapid and open access to data. The six principles agreed upon by key stakeholders at the summit addressed issues surrounding (1) timing, (2) comprehensiveness, (3) format, (4) deposition to repositories, (5) quality metrics, and (6) responsibility for proteomics data release. This summit report explores various approaches to develop a framework of data release and sharing principles that will most effectively fulfill the needs of the funding agencies and the research community. \n",
      "\n",
      "------- \n",
      "\n",
      "The environment drives microbial trait variability in aquatic habitats \n",
      "\n",
      "['Beier, S.', 'Andersson, Anders', 'Galand, P.', 'Hochart, C.', 'Logue, J.', 'McMahon, K.', 'Bertilsson, S.'] \n",
      "\n",
      "A prerequisite to improve the predictability of microbial community dynamics is to understand the mechanisms of microbial assembly. To study factors that contribute to microbial community assembly, we examined the temporal dynamics of genes in five aquatic metagenome time-series, originating from marine offshore or coastal sites and one lake. With this trait-based approach we expected to find gene-specific patterns of temporal allele variability that depended on the seasonal metacommunity size of carrier-taxa and the variability of the milieu and the substrates to which the resulting proteins were exposed. In more detail, we hypothesized that a larger seasonal metacommunity size would result in increased temporal variability of functional units (i.e., gene alleles), as shown previously for taxonomic units. We further hypothesized that multicopy genes would feature higher temporal variability than single-copy genes, as gene multiplication can result from high variability in substrate quality and quantity. Finally, we hypothesized that direct exposure of proteins to the extracellular environment would result in increased temporal variability of the respective gene compared to intracellular proteins that are less exposed to environmental fluctuations. The first two hypotheses were confirmed in all data sets, while significant effects of the subcellular location of gene products was only seen in three of the five time-series. The gene with the highest allele variability throughout all data sets was an iron transporter, also representing a target for phage infection. Previous work has emphasized the role of phage–prokaryote interactions as a major driver of microbial diversity. Our finding therefore points to a potentially important role of iron transporter-mediated phage infections for the assembly and maintenance of diversity in aquatic prokaryotes. \n",
      "\n",
      "------- \n",
      "\n",
      "Coolability of particulate beds in severe accidents : Status and remaining uncertainties \n",
      "\n",
      "['Buerger, M.', 'Buck, M.', 'Pohlner, G.', 'Rahman, S.', 'Kulenovic, R.', 'Fichot, F.', 'Ma, Weimin', 'Miettinen, J.', 'Lindholm, I.', 'Atkhen, K.'] \n",
      "\n",
      "Particulate debris beds may form during different stages of a severe core melt accident; e.g. in the degrading hot core, due to thermal stresses during reflooding, in the lower plenum, by melt flow from the core into water in the lower head, and in the cavity by melt flow out of a failing RPV into a wet cavity. Deep water pools in the cavity are used in Nordic BWRs as an accident management measure aiming at particulate debris formation and coolability. It has been elaborated in the joint work of the European Severe Accident Research Network (SARNET) in Work Package (WP) 11.1 that coolability of particulate debris, reflooding of hot debris as well as boil-off under decay heat (long-term coolability), is strongly favoured by 2D/3D effects in beds with non-homogeneous structure and shape. Especially, water inflow from the sides and via bottom regions strongly improves coolability as compared to 1D situations with top flooding, the latter being in the past the basis of analyses on coolability. Data from experiments included in the SARNET network (DEBRIS at IKE and STYX at VTT) and earlier ones (e.g. POMECO at KTH) have been used to validate key constitutive laws in 2D codes as WABE (IKE) and ICARE/CATHARE (IRSN), especially concerning flow friction and heat transfer. Major questions concern the need of the explicit use of interfacial friction to adequately treat the various flow situations in a unified approach, as well as the adequate characterization of realistic debris composed of irregularly shaped particles of different sizes. joint work has been supported by transfer of the WABE code to KTH and VTT. Concerning realistic debris, the formation from breakup of melt jets in water is investigated in the DEFOR experiments at KTH. Present results indicate that porosities in the debris might be much higher than previously assumed, which would strongly support attainment of coolability. Calculations have been performed with IKEJET/IKEMIX describing jet breakup, mixing and settling of resulting particles. Models about debris bed formation and porosity are developed at KTH. The codes have been applied to reactor conditions for analysing the potential for coolability in the different phases of a severe accident. Calculations have been performed with WABE (MEWA) implemented in ATHLET-CD and with ICARE/ICATHARE for degraded cores and debris beds in the lower plenum, under reflooding and boil-off. Ex-vessel situations have also been analysed. Strong effects of lateral water inflow and cooling by steam in hot areas have been demonstrated. In support, some typical basic configurations have been analysed, e.g. configurations with downcomers considered as possible AM measures. Melt pool formation or coolability of particulate debris is a major issue concerning melt retention in the core and the lower head. Present conclusions from those analyses for adequate modelling in ASTEC are outlined as well as remaining uncertainties. Experimental and analysis efforts and respective continued joint actions are discussed, which are needed to reach resolution of the coolability issue. \n",
      "\n",
      "------- \n",
      "\n",
      "Post-metaphase correction of aberrant kinetochore-microtubule attachments in mammalian eggs \n",
      "\n",
      "['Kouznetsova, Anna', 'Kitajima, Tomoya', 'Brismar, Hjalmar', 'Höög, Christer'] \n",
      "\n",
      "The accuracy of the two sequential meiotic divisions in oocytes is essential for creating a haploid gamete with a normal chromosomal content. Here, we have analysed the 3D dynamics of chromosomes during the second meiotic division in live mouse oocytes. We find that chromosomes form stable kinetochore-microtubule attachments at the end of prometaphase II stage that are retained until anaphase II onset. Remarkably, we observe that more than 20% of the kinetochore-microtubule attachments at the metaphase II stage are merotelic or lateral. However, II are found to lag at the spindle equator and &lt; 10% of the laggards missegregate and give rise to aneuploid gametes. Our results demonstrate that aberrant kinetochore-microtubule attachments are not corrected at the metaphase stage of the second meiotic division. Thus, the accuracy of the chromosome segregation process in mouse oocytes during meiosis II is ensured by an efficient correction process acting at the anaphase stage. \n",
      "\n",
      "------- \n",
      "\n",
      "Genetic tumor archeology : microdissection and genetic heterogeneity in squamous and basal cell carcinoma \n",
      "\n",
      "['Backvall, H.', 'Asplund, A.', 'Gustafsson, A.', 'Sivertsson, A.', 'Lundeberg, Joakim', 'Ponten, F.'] \n",
      "\n",
      "Carcinogenesis is a multi-step series of somatic genetic events. The complexity of this multi-hit process makes it difficult to determine each single event and the definitive outcome of such events. To investigate the genetic alterations in cancer-related genes, sensitive and reliable detection methods are of major importance for generating relevant results. Another critical issue is the quality of starting material which largely affects the outcome of the analysis. Microdissection of cells defined under the microscope ensures a selection of representative material for subsequent genetic analysis. Skin cancer provides an advantageous model for studying the development of cancer. Detectable lesions occur early during tumor progression, facilitating molecular analysis of the cell populations from both preneoplastic and neoplastic lesions. Alterations of the p53 tumor suppressor gene are very common in non-melanoma skin cancer, and dysregulation of p53 pathways appear to be an early event in the tumor development. A high frequency of epidermal p53 clones has been detected in chronically sun-exposed skin. The abundance of clones containing p53 mutated keratinocytes adjacent to basal cell (BCC) and squamous cell carcinoma (SCC) suggests a role in human skin carcinogenesis. Studies using p53 mutations as a clonality marker have suggested a direct link between actinic keratosis, SCC in situ and invasive SCC. Microdissection-based studies have also shown that different parts of individual BCC tumors can share a common p53 mutation yet differ with respect to additional alterations within the p53 gene, consistent with subclonal development within tumors. Here, we present examples of using well-defined cell populations, including single cells, from complex tissue in combination with molecular tools to reveal features involved in skin carcinogenesis. \n",
      "\n",
      "------- \n",
      "\n",
      "A validated liquid chromatography tandem mass spectrometry method for quantification of erlotinib, OSI-420 and didesmethyl erlotinib and semi-quantification of erlotinib metabolites in human plasma \n",
      "\n",
      "['Svedberg, Anna', 'Green, Henrik', 'Vikström, Anders', 'Lundeberg, Joakim', 'Vikingsson, Svante'] \n",
      "\n",
      "A liquid chromatography tandem mass spectrometry method was developed and validated for quantification of erlotinib and its metabolites in human plasma. The method is suitable for therapeutic drug monitoring and pharmacokinetic studies. The substances were extracted using protein precipitation, separated on a BEH XBridge C18 column (100 x 2.1 mm, 1.7 mu m) by gradient elution at 0.7 mL/min of acetonitrile and 5 mM ammonium acetate. The concentration was determined using a Waters Xevo triple quadrupole mass spectrometer in a multi reaction monitoring mode. The total run time was 7 min. Deuterated erlotinib and OSI-597 were used as internal standard for erlotinib and its metabolites, respectively. Erlotinib, OSI-420 and didesmethyl erlotinib were quantified in the concentration range 25-5000 ng/mL, 0.5-500 ng/mL and 0.15-10 ng/mL, respectively. Precision and accuracy was &lt;14% except for OSI-420 at LLOQ (17%). Extraction recovery was above 89%, 99% and 89% for erlotinib, OSI-420 and didesmethyl erlotinib, respectively. The human liver microsomes generated 14 metabolites, three of them not previously reported. Twelve metabolites were measured semi-quantitatively and validated with respect to selectivity, precision and stability. \n",
      "\n",
      "------- \n",
      "\n",
      "Nitrogenase Gene Amplicons from Global Marine Surface Waters Are Dominated by Genes of Non-Cyanobacteria \n",
      "\n",
      "['Farnelid, Hanna', 'Andersson, Anders', 'Bertilsson, Stefan', 'Abu Al-Soud, Waleed', 'Hansen, Lars', 'Sørensen, Søren', 'Steward, Grieg', 'Hagstöom, Åke', 'Riemann, Lasse'] \n",
      "\n",
      "Cyanobacteria are thought to be the main N-2-fixing organisms (diazotrophs) in marine pelagic waters, but recent molecular analyses indicate that non-cyanobacterial diazotrophs are also present and active. Existing data are, however, restricted geographically and by limited sequencing depths. Our analysis of 79,090 nitrogenase (nifH) PCR amplicons encoding 7,468 unique proteins from surface samples (ten DNA samples and two RNA samples) collected at ten marine locations worldwide provides the first in-depth survey of a functional bacterial gene and yield insights into the composition and diversity of the nifH gene pool in marine waters. Great divergence in nifH composition was observed between sites. Cyanobacteria-like genes were most frequent among amplicons from the warmest waters, but overall the data set was dominated by nifH sequences most closely related to non-cyanobacteria. Clusters related to Alpha-, Beta-, Gamma-, and Delta-Proteobacteria were most common and showed distinct geographic distributions. Sequences related to anaerobic bacteria (nifH Cluster III) were generally rare, but preponderant in cold waters, especially in the Arctic. Although the two transcript samples were dominated by unicellular cyanobacteria, 42% of the identified non-cyanobacterial nifH clusters from the corresponding DNA samples were also detected in cDNA. The study indicates that non-cyanobacteria account for a substantial part of the nifH gene pool in marine surface waters and that these genes are at least occasionally expressed. The contribution of non-cyanobacterial diazotrophs to the global N-2 fixation budget cannot be inferred from sequence data alone, but the prevalence of non-cyanobacterial nifH genes and transcripts suggest that these bacteria are ecologically significant. \n",
      "\n",
      "------- \n",
      "\n",
      "Crystal structure of the polyextremophilic alpha-amylase AmyB from Halothermothrix orenii : Details of a productive enzyme-substrate complex and an N domain with a role in binding raw starch \n",
      "\n",
      "['Tan, Tien-Chye', 'Mijts, Benjamin', 'Swaminathan, Kunchithapadam', 'Patel, Bharat', 'Divne, Christina'] \n",
      "\n",
      "The gene for a membrane-bound, halophilic, and thermostable alpha-amylase, AmyB, from Halothermothrix orenii was cloned and sequenced. The crystal structure shows that, in addition to the typical domain organization of family 13 glycoside hydrolases, AmyB carries an additional N-terminal domain (N domain) that forms a large groove-the N-C groove some 30 angstrom away from the active site. The structure of AmyB with the inhibitor acarbose at 1.35 angstrom resolution shows that a nonasaccharide has been synthesized through successive transglycosylation reactions of acarbose. Unexpectedly, in a complex of wild-type AmyB with alpha-cyclodextrin and maltoheptaose at 2.2 angstrom resolution, a maltotetraose molecule is bound in subsites -1 to +3, spanning the cleavage point at -1 / + 1, with the -1 glucosyl residue present as a S-2(o) skew boat. This wild-type AmyB complex was obtained in the presence of a large excess of substrate, a condition under which it is possible to capture Michaelis complexes, which may explain the observed binding across -1/+ 1 and ring distortion. We observe three methionine side chains that serve as '' binding platforms '' for glucosyl rings in AmyB, a seemingly rare occurrence in carbohydrate-binding proteins. The structures and results from the biochemical characterization of AmyB and AmyB lacking the N domain show that the N domain increases binding of the enzyme to raw starch. Furthermore, theoretical modeling suggests that the N-C groove can accommodate, spatially and chemically, large substrates such as A-starch. \n",
      "\n",
      "------- \n",
      "\n",
      "TcellSubC : An Atlas of the Subcellular Proteome of Human T Cells \n",
      "\n",
      "['Joshi, Rubin', 'Karolinska Univ Hosp, Stockholm', 'Sci Life Lab, Stockholm', 'Stadler, Charlotte', 'Lehmann, Robert', 'Lehtio, Janne', 'Tegner, Jesper', 'Karolinska Univ Hosp, Stockholm', 'Sci Life Lab, Stockholm', 'King Abdullah Univ Sci & Technol, Comp', 'Schmidt, Angelika', 'Karolinska Univ Hosp, Stockholm', 'Sci Life Lab, Stockholm', 'Ludwig Maximilians Univ Munchen, Inst', 'Vesterlund, Mattias'] \n",
      "\n",
      "We have curated an in-depth subcellular proteomic map of primary human CD4+ T cells, divided into cytosolic, nuclear and membrane fractions generated by an optimized fractionation and HiRIEF-LC-MS/MS workflow for limited amounts of primary cells. The subcellular proteome of T cells was mapped under steady state conditions, as well as upon 15 min and 1 h of T cell receptor (TCR) stimulation, respectively. We quantified the subcellular distribution of 6,572 proteins and identified a subset of 237 potentially translocating proteins, including both well-known examples and novel ones. Microscopic validation confirmed the localization of selected proteins with previously known and unknown localization, respectively. We further provide the data in an easy-to-use web platform to facilitate re-use, as the data can be relevant for basic research as well as for clinical exploitation of T cells as therapeutic targets. \n",
      "\n",
      "------- \n",
      "\n",
      "Trade-Offs between phosphorous discharge and hydropower production using reservoir regulation \n",
      "\n",
      "['Zmijewski, Nicholas', 'Wörman, Anders'] \n",
      "\n",
      "The management of water flow using reservoirs for the purpose of hydropower production can have significant impacts on solute transport. Reducing the phosphorous load to downstream areas can be of great importance for aquatic habitats associated with recipient waters, as well as for the overall health of coastal and estuary areas. This study demonstrates a combined operation of a network of reservoirs with the objectives of hydropower production and improvement of water quality in downstream areas using a multiobjective approach and incorporating the transport of phosphorous in the watershed. A genetic optimization method was used to determine the trade-off between power production and mitigation of phosphorous discharge in the reservoir network of the Dalälven River, Sweden. The phosphorous export can be reduced (-5%) with a limited loss of power production (-4.3%), but further reduction is associated with a significant loss in total power production. The trade-off between the two objectives is determined using Pareto diagrams and examined for varying particulate conditions. \n",
      "\n",
      "------- \n",
      "\n",
      "Arabidopsis acyl-acyl carrier protein synthetase AAE15 with medium chain fatty acid specificity is functional in cyanobacteria \n",
      "\n",
      "['Kaczmarzyk, Danuta', 'Hudson, Elton', 'Fulda, Martin'] \n",
      "\n",
      "Cyanobacteria are potential hosts for the biosynthesis of oleochemical compounds. The metabolic precursors for such compounds are fatty acids and their derivatives, which require chemical activation to become substrates in further conversion steps. We characterized the acyl activating enzyme AAE15 of Arabidopsis encoded by At4g14070, which is a homologue of a cyanobacterial acyl-ACP synthetase (AAS). We expressed AAE15 in insect cells and demonstrated its AAS activity with medium chain fatty acid (C10-C14) substrates in vitro. Furthermore, we used AAE15 to complement a Synechocystis aas deletion mutant and showed that the new strain preferentially incorporates supplied medium chain fatty acids into internal lipid molecules. Based on this data we propose that AAE15 can be utilized in metabolic engineering strategies for cyanobacteria that aim to produce compounds based on medium chain fatty acids. \n",
      "\n",
      "------- \n",
      "\n",
      "Global parameters sensitivity analysis of modeling water, energy and carbon exchange of an arid agricultural ecosystem \n",
      "\n",
      "['Wu, M.', 'Ran, Y.', 'Jansson, Per-Erik', 'Chen, P.', 'Tan, X.', 'Zhang, W.'] \n",
      "\n",
      "Agricultural ecosystems are important for regulating terrestrial hydrological and carbon cycles. Hydrological and carbon processes in agricultural ecosystem models are complex due to interactions between parameters. It is therefore crucial to identify parameter sensitivity before a process-based model is applied for simulations and predictions of water, energy and carbon fluxes in agricultural ecosystems. In this study, we investigated the sensitivity and equifinality of the CoupModel parameters in modeling an arid agricultural ecosystem in northwestern China. In total, 27 model parameters were analyzed using a global parameters sensitivity analysis approach and a combination of multiple in situ and remotely sensed data sets. Among the five major model processes, we found that the energy balance process account for much of the importance in the model, followed by soil hydrology, plant growth, soil heat, and soil carbon processes. Meanwhile, parameters from the plant growth process exhibited higher equifinalities than other processes. We found that net ecosystem exchange (NEE) is controlled by soil heat, soil hydrology and energy balance processes, which is mainly due to a high equifinality (0.91) between the parameters g max (maximal stomatal conductance) and V cmax (maximal carboxylation rate). The equifinalities between different parameters result in a trade-off in model performance metrics (i.e. determination coefficient R 2 and mean error ME) in the water, energy and carbon balance simulations. We revealed that daytime and yearly accumulated eddy fluxes (sensible heat H s , latent heat LE and NEE) can constrain the model parameters better. Remotely sensed data were also promising as additional constraints on soil water contents and energy fluxes. This study introduced a systematic global parameter sensitivity analysis approach together with the equifinality identification in an ecosystem model. The approach proposed here is applicable to other studies and the equifinalities detected in this study can be important implications for modelling arid agricultural ecosystems. Additional exploration on remotely sensed data in constraining the model from different aspects are highly recommended in modeling agricultural ecosystems. © 2019 Elsevier B.V. \n",
      "\n",
      "------- \n",
      "\n",
      "Orientation effect on heat transfer coefficient of a downward surface for flow boiling in a rectangular channel under low flow rate \n",
      "\n",
      "['Gong, S.', 'Mei, Y.', 'Amin, M.', 'Zhang, B.', 'Ma, Weimin'] \n",
      "\n",
      "Natural convection boiling in channel with Arc-Shaped will be encountered in the IVR-ERVC (In-Vessel Retention measure by External Reactor Vessel Cooling) system in nuclear power plant under severe accident. The flow and heat transfer characters in this situation is simulated by flow boiling of deionized water in an inclined rectangular channel under low flow rates. This paper aims to separate various parameters (such as orientation, mass flow rate and inlet quality, etc.) to investigate their individual effects on heat transfer coefficient (HTC) in a rectangular channel with cross section of 17 mm × 10 mm. By using a preheater at the inlet of the rectangular channel, the inlet quality could be controlled and the two-phase flow situation could be observed before the fluids entering into the main heater region on one side of the channel wall in downstream. Thus the characteristics of HTC on the main heater could be investigated at different flow patterns. The channel orientations vary from 15 to 90°, the mass flow rates vary from 110 to 288 kg/(m2s) and the qualities vary from 0.003 to 0.036, respectively. Experimental results show that the mass flow rate and quality effects on the HTC are very weak in this study. However, the orientation angle effect on HTC shows an transition region within 45°~60°, while it slowly changes when the orientation angle is smaller than 45° and bigger than 60°. Such tendency could be well formulated by the error function. Compared with different empirical formulas of saturated boiling HTC, it is found that the Liu &amp;amp; Winterton correlation can well predict the experimental HTC results in 90° orientation channel. Based on such correlation and coupled with the error function, a new model was developed by considering the orientation effect, which has an error of ±15% comparing with the experimental data. \n",
      "\n",
      "------- \n",
      "\n",
      "Super-resolution microscopy reveals gamma-secretase at both sides of the neuronal synapse \n",
      "\n",
      "['Schedin-Weiss, Sophia', 'Caesar, Ina', 'Winblad, Bengt', 'Blom, Hans', 'Tjernberg, Lars'] \n",
      "\n",
      "The transmembrane protein assembly gamma-secretase is a key protease in regulated intramembrane processing (RIP) of around 100 type-1 transmembrane proteins. Importantly, it has a pathological role in Alzheimer disease (AD) as it generates the neurotoxic amyloid beta-peptide from the amyloid precursor protein (APP). Studies on gamma-secretase location are therefore crucial both from a biological and a therapeutic perspective. Despite several years of efforts in many laboratories, it is not clear where in the neuron gamma-secretase exerts it's activities. Technical challenges include the fact that the active enzyme contains four protein components and that most subcellular compartments cannot be spatially resolved by traditional light microscopy. Here, we have used a powerful combination of the two nanoscopy techniques STORM and STED microscopy to visualize the location of gamma-secretase in neurons using an active-site specific probe, with a focus on the synapse. We show that gamma-secretase is present in both the pre-and postsynaptic compartments. We further show that the enzyme is enriched very close to the synaptic cleft in the postsynaptic membrane, as well as to NMDA receptors, demonstrating that gamma-secretase is present in the postsynaptic plasma membrane. Importantly, the expression of gamma-secretase increased in the pre-and postsynaptic compartments with the size of the synapse, suggesting a correlation between gamma-secretase activity and synapse maturation. Thus, our data shows the synaptic location with high precision in three dimensions and settles the long-lasting debate on the synaptic location of gamma-secretase. \n",
      "\n",
      "------- \n",
      "\n",
      "NeuroAED : Towards Efficient Abnormal Event Detection in Visual Surveillance With Neuromorphic Vision Sensor \n",
      "\n",
      "['Chen, Guang', 'Tech Univ Munich, Chair', 'Chinese Acad Sci, State', 'Liu, Peigen', 'Liu, Zhengfa', 'Tang, Huajin', 'Hong, Lin', 'Dong, Jinhu', 'Conradt, Jörg', 'Knoll, Alois'] \n",
      "\n",
      "Abnormal event detection is an important task in research and industrial applications, which has received considerable attention in recent years. Existing methods usually rely on standard frame-based cameras to record the data and process them with computer vision technologies. In contrast, this paper presents a novel neuromorphic vision based abnormal event detection system. Compared to the frame-based camera, neuromorphic vision sensors, such as Dynamic Vision Sensor (DVS), do not acquire full images at a fixed frame rate but rather have independent pixels that output intensity changes (called events) asynchronously at the time they occur. Thus, it avoids the design of the encryption scheme. Since events are triggered by moving edges on the scene, DVS is a natural motion detector for the abnormal objects and automatically filters out any temporally-redundant information. Based on this unique output, we first propose a highly efficient method based on the event density to select activated event cuboids and locate the foreground. We design a novel event-based multiscale spatio-temporal descriptor to extract features from the activated event cuboids for the abnormal event detection. Additionally, we build the NeuroAED dataset, the first public dataset dedicated to abnormal event detection with neuromorphic vision sensor. The NeuroAED dataset consists of four sub-datasets: Walking, Campus, Square, and Stair dataset. Experiments are conducted based on these datasets and demonstrate the high efficiency and accuracy of our method. \n",
      "\n",
      "------- \n",
      "\n",
      "Analysis of Candidate Genes for Lineage-Specific Expression Changes in Humans and Primates \n",
      "\n",
      "['Lindskog, Cecilia', 'Kuhlwilm, Martin', 'Davierwala, Armaity', 'Fu, Ning', 'Hegde, Geeta', 'Uhlén, Mathias', 'Navani, Sanjay', 'Paeaebo, Svante', 'Ponten, Fredrik'] \n",
      "\n",
      "RUNX2, a gene involved in skeletal development, has previously been shown to be potentially affected by positive selection during recent human evolution. Here we have used antibody-based proteomics to characterize potential differences in expression patterns of RUNX2 interacting partners during primate evolution. Tissue microarrays consisting of a large set of normal tissues from human and macaque were used for protein profiling of 50 RUNX2 partners with immunohistochemistry. Eleven proteins (AR, CREBBP, EP300, FGF2, HDAC3, JUN, PRKD3, RUNX1, SATB2, TCF3, and YAP1) showed differences in expression between humans and macaques. These proteins were further profiled in tissues from chimpanzee, gorilla, and orangutan, and the corresponding genes were analyzed with regard to genomic features. Moreover, protein expression data were compared with previously obtained RNA sequencing data from six different organs. One gene (TCF3) showed significant expression differences between human and macaque at both the protein and RNA level, with higher expression in a subset of germ cells in human testis compared with macaque. In conclusion, normal tissues from macaque and human showed differences in expression of some RUNX2 partners that could be mapped to various defined cell types. The applied strategy appears advantageous to characterize the consequences of altered genes selected during evolution. \n",
      "\n",
      "------- \n",
      "\n",
      "Fluidic Injection Scenarios for Shock Pattern Manipulation in Exhausts \n",
      "\n",
      "['Semlitsch, Bernhard', 'Mihaescu, Mihai'] \n",
      "\n",
      "By numerically screening internal fluidic injection scenarios for the manipulation of the double diamond shock pattern in convergent-divergent nozzle exhausts, the individual importance of design parameters is demonstrated. It is found that the evolving shock pattern is sensitive to the injection location, whereas the persistence of the induced counterrotating vortex pairs is primarily governed by the injection pressure. Injection close to the nozzle exit generates secondary vortical structures, amplifying the fluctuations in the nozzle vicinity. \n",
      "\n",
      "------- \n",
      "\n",
      "Cellobiose dehydrogenase - A flavocytochrome from wood-degrading, phytopathogenic and saprotropic fungi \n",
      "\n",
      "['Zamocky, M.', 'Ludwig, R.', 'Peterbauer, C.', 'Hallberg, B.', 'Divne, Christina', 'Nicholls, P.', 'Haltrich, D.'] \n",
      "\n",
      "Cellobiose dehydrogenase, the only currently known extracellular flavocytochrome. is formed not only by a number of wood-degrading but also by various phytopathogenic fungi. This inducible enzyme participates in early events of lignocellulose degradation, as investigated in several basidiomycete fungi at the transcriptional and translational level. However, its role in the ascomycete fungi is not yet obvious. Comprehensive sequence analysis of CDH-encoding genes and their translational products reveals significant sequence similarities along the entire sequences and also a common domain architecture. All known cellobiose dehydrogenases fall into two related subgroups. Class-I members are represented by sequences from basidiomycetcs whereas class-II comprises longer, more complex sequences from ascomycete fungi. Cellobiose dehydrogenase is typically a monomeric protein consisting of two domains joined by a protease-sensitive linker region. Each larger (dehydrogenase) domain is flavin-associated while the smaller (cytochrome) domains are haem-binding. The latter shorter domains are unique sequence motifs for all currently known flavocytochromes. Each cytochrome domain of CDH can bind a single haem b as prosthetic group. The larger dehydrogenase domain belongs to the glucose-methanol-choline (GMC) oxidoreductase superfamily - a widespread flavoprotein evolutionary line. The larger domains can be further divided into a flavin-binding subdomain and a substrate-binding subdomain. In addition, the class-II (but not class-I) proteins can possess a short cellulose-binding module of type I at their C-termini. All the cellobiose dehydrogenases oxidise cellobiose, cellodextrins, and lactose to the corresponding lactones using a wide spectrum of different electron acceptors. Their flexible specificity serves as a base for the development of possible biotechnological applications. \n",
      "\n",
      "------- \n",
      "\n",
      "A conversational robotic lab assistant for automated microfluidic 3d microtissue production \n",
      "\n",
      "['Langer, Krzysztof', 'Jernström, Sandra', 'Mikkonen, Piia', 'Östling, Päiivi', 'Seashore-Ludlow, Brinton', 'Jönsson, Håkan'] \n",
      "\n",
      "The future of life science is linked to automation and microfluidics. Here we present a robotic lab assistant, a general automation platform for droplet microfluidics including a conversational mobile interface. We demonstrate the automated production of human cancer microtissues in droplets at a throughput of 85000 spheroids per microfluidic circuit per hour. The capability of automated spheroid generation is directly applicable to precision medicine and drug screening. Multiple cell lines were successfully tested, including cancer cell lines, co-cultures, and primary cells. The 3D-microtissues/spheroids were automatically assembled-incubated-retrieved with high viability for further drug screening analysis - the platform interfaces with standard labware. \n",
      "\n",
      "------- \n",
      "\n",
      "ASTEC extension to other reactor types than Generation II PWR \n",
      "\n",
      "['Van Dorsselaere, J.P.', 'Schwinges, B.', 'Buck, M.', 'Ma, Weimin', 'Constantin, M.', 'Jancovic, J.', 'Ratel, G.'] \n",
      "\n",
      "The initial IRSN-GRS requirements for the development of the ASTEC European integral codecovered all present and future PWR, VVER and BWR. After 4 years and a half of SARNET activities, theASTEC V1 code is fully applicable to all severe accident scenarios, at power operation, in Gen.II PWR andVVER. Partners performed substantial code assessment tasks on both VVER-440 and 1000 with good resultson validation (such as PACTEL) and benchmarking with other codes on plant sequences.In the frame of preparation of the new series of ASTEC V2 versions that will account for the needs ofevolution expressed by the SARNET partners, the extension to Gen.III PWR is under way. The 1stversionV2.0, planned for march 09, will be applicable to the EPR, in particular its external core-catcher and to newPWR designs with In-Vessel Melt Retention (IVMR), the latter thanks to CEA work. IVS calculations onVVER-440/V213 showed the ASTEC V1 capabilities to evaluate the IVMR possibilities: these capabilitieswill be improved in the future through a coupled analysis of in-vessel corium and cavity cooling circuit.For BWR applications, the KTH ranking of needs of model adaptations was reviewed by GRS. Theseadaptations mainly concern the Reactor Cooling System (RCS) thermal-hydraulics (internal pumps, drier,separator...) and the core degradation (canisters, control rod guide tubes, penetrations in lower head...). Forcore degradation, the scoping application test on the CORA-18 experiment using the current ASTEC V1models (for adaptation to absorber blades and canisters) showed promising results. The GRS benchmark withthe COCOSYS GRS code showed the applicability of the CPA containment module. For ex-vesselphenomena, the only missing model is the formation of a debris bed during corium slump from lower headinto a flooded cavity after lower head failure and its coolability/erosion behaviour with core concreteinteraction under flooded conditions. All other current V1 models can be used, sometimes with minoradaptations or further need of validation, as for iodine behaviour in containment at temperatures up to 1000Kand the possible decomposition of metal iodides to gaseous iodine at these high temperatures in atmosphere.Detailed specifications will be written soon to prepare the development and validation task planned inSARNET2 with the same partners.For CANDU reactors applications, the priority concerns the core degradation due to the peculiarity ofthe vessel geometry (horizontal core, pressure tubes, calandria...). The exploratory plant applications haveshown that all other current models can be used but tests are still necessary to conclude on models of PHT(Primary Heat Transfer) thermal-hydraulics. INR work takes benefit of the AECL knowledge of MAAP4-CANDU models. A significant benefit is foreseen from the BARC (India) work on core degradation modelsin the frame of IRSN-BARC bilateral collaboration. \n",
      "\n",
      "------- \n",
      "\n",
      "Rapid production and recovery of cell spheroids by automated droplet microfluidics \n",
      "\n",
      "['Langer, Krzysztof', 'Jönsson, Håkan'] \n",
      "\n",
      "Droplet microfluidics enables high throughput cell processing, analysis and screening by miniaturizing the reaction vessels to nano- or pico-liter water-in oil droplets, but like many other microfluidic formats, droplet microfluidics have not been interfaced with or automated by laboratory robotics. Here we demonstrate automation of droplet microfluidics based on an inexpensive liquid handling robot for the automated production of human scaffold-free cell spheroids, using pipette actuation and interfacing the pipetting tip with a droplet generating microfluidic chip. In this chip we produce highly mono-disperse 290μm droplets with diameter CV of 1.7%. By encapsulating cells in these droplets, we produce cell spheroids in droplets and recover them to standard formats at a throughput of 85000 spheroids per microfluidic circuit per hour. The viability of the cells in spheroids remains high after recovery only decreased by 4% starting from 96% after 16 hours incubation in nanoliter droplets. Scaffold-free cell spheroids and 3D tissue constructs recapitulate many aspects of functional human tissue more accurately than 2D or single cell cultures, but assembly methods for spheroids, e.g. hanging drop micro-plates, has had limited throughput. The increased throughput and decreased cost of our method enables spheroid production at the scale needed for lead discovery drug screening and approaches the cost where these micro tissues could be used as building blocks for organ scale regenerative medicine. \n",
      "\n",
      "------- \n",
      "\n",
      "Rapid production and recovery of cell spheroids by automated droplet microfluidics \n",
      "\n",
      "['Langer, Krzysztof', 'Jönsson, Håkan'] \n",
      "\n",
      "Droplet microfluidics enables high throughput cell processing, analysis and screening by miniaturizing the reaction vessels to nano- or pico-liter water-in oil droplets, but like many other microfluidic formats, droplet microfluidics have not been interfaced with or automated by laboratory robotics. Here we demonstrate automation of droplet microfluidics based on an inexpensive liquid handling robot for the automated production of human scaffold-free cell spheroids, using pipette actuation and interfacing the pipetting tip with a droplet generating microfluidic chip. In this chip we produce highly mono-disperse 290μm droplets with diameter CV of 1.7%. By encapsulating cells in these droplets, we produce cell spheroids in droplets and recover them to standard formats at a throughput of 85000 spheroids per microfluidic circuit per hour. The viability of the cells in spheroids remains high after recovery only decreased by 4% starting from 96% after 16 hours incubation in nanoliter droplets. Scaffold-free cell spheroids and 3D tissue constructs recapitulate many aspects of functional human tissue more accurately than 2D or single cell cultures, but assembly methods for spheroids, e.g. hanging drop micro-plates, has had limited throughput. The increased throughput and decreased cost of our method enables spheroid production at the scale needed for lead discovery drug screening and approaches the cost where these micro tissues could be used as building blocks for organ scale regenerative medicine. \n",
      "\n",
      "------- \n",
      "\n",
      "Community-wide analysis of microbial genome sequence signatures \n",
      "\n",
      "['Dick, G', 'Andersson, Anders', 'Baker, B', 'Simmons, S', 'Thomas, B', 'Yelton, A', 'Banfield, J'] \n",
      "\n",
      "BACKGROUND: Analyses of DNA sequences from cultivated microorganisms have revealed genome-wide, taxa-specific nucleotide compositional characteristics, referred to as genome signatures. These signatures have far-reaching implications for understanding genome evolution and potential application in classification of metagenomic sequence fragments. However, little is known regarding the distribution of genome signatures in natural microbial communities or the extent to which environmental factors shape them. RESULTS: We analyzed metagenomic sequence data from two acidophilic biofilm communities, including composite genomes reconstructed for nine archaea, three bacteria, and numerous associated viruses, as well as thousands of unassigned fragments from strain variants and low-abundance organisms. Genome signatures, in the form of tetranucleotide frequencies analyzed by emergent self-organizing maps, segregated sequences from all known populations sharing &lt; 50 to 60% average amino acid identity and revealed previously unknown genomic clusters corresponding to low-abundance organisms and a putative plasmid. Signatures were pervasive genome-wide. Clusters were resolved because intra-genome differences resulting from translational selection or protein adaptation to the intracellular (pH approximately 5) versus extracellular (pH approximately 1) environment were small relative to inter-genome differences. We found that these genome signatures stem from multiple influences but are primarily manifested through codon composition, which we propose is the result of genome-specific mutational biases. CONCLUSIONS: An important conclusion is that shared environmental pressures and interactions among coevolving organisms do not obscure genome signatures in acid mine drainage communities. Thus, genome signatures can be used to assign sequence fragments to populations, an essential prerequisite if metagenomics is to provide ecological and biochemical insights into the functioning of microbial communities. \n",
      "\n",
      "------- \n",
      "\n",
      "Simultaneous Identification of Duplications and Lateral Gene Transfers \n",
      "\n",
      "['Tofigh, Ali', 'Hallett, Mikael', 'Lagergren, Jens'] \n",
      "\n",
      "The incongruency between a gene tree and a corresponding species tree can be attributed to evolutionary events such as gene duplication and gene loss. This paper describes a combinatorial model where a so-called DTL-scenario is used to explain the differences between a gene tree anda corresponding species tree taking into account gene duplications, gene losses, and lateral genetransfers (also known as horizontal gene transfers). The reasonable biological constraint that a lateralgene transfer may only occur between contemporary species leads to the notion of acyclic DTLscenarios.Parsimony methods are introduced by defining appropriate optimization problems. Weshow that finding most parsimonious acyclic DTL-scenarios is NP-complete. However, by droppingthe condition of acyclicity, the problem becomes tractable, and we provide a dynamic programmingalgorithm as well as a fixed-parameter-tractable algorithm for finding most parsimonious DTLscenarios. \n",
      "\n",
      "------- \n",
      "\n",
      "Freshwater bacterioplankton richness in oligotrophic lakes depends on nutrient availability rather than on species-area relationships \n",
      "\n",
      "['Logue, Jürg', 'Langenheder, Silke', 'Andersson, Anders', 'Bertilsson, Stefan', 'Drakare, Stina', 'Lanzén, Anders', 'Lindström, Eva'] \n",
      "\n",
      "A central goal in ecology is to grasp the mechanisms that underlie and maintain biodiversity and patterns in its spatial distribution can provide clues about those mechanisms. Here, we investigated what might determine the bacterioplankton richness (BR) in lakes by means of 454 pyrosequencing of the 16S rRNA gene. We further provide a BR estimate based upon a sampling depth and accuracy, which, to our knowledge, are unsurpassed for freshwater bacterioplankton communities. Our examination of 22 669 sequences per lake showed that freshwater BR in fourteen nutrient-poor lakes was positively influenced by nutrient availability. Our study is, thus, consistent with the finding that the supply of available nutrients is a major driver of species richness; a pattern that may well be universally valid to the world of both micro- and macro-organisms. We, furthermore, observed that BR increased with elevated landscape position, most likely as a consequence of differences in nutrient availability. Finally, BR decreased with increasing lake and catchment area that is negative species-area relationships (SARs) were recorded; a finding that re-opens the debate about whether positive SARs can indeed be found in the microbial world and whether positive SARs can in fact be pronounced as one of the few 'laws' in ecology. \n",
      "\n",
      "------- \n",
      "\n",
      "Effect of neutron source on the incineration of light Water Reactors waste by prismatic high temperature reactors \n",
      "\n",
      "['Zakova, Jitka', 'Talamo, A.'] \n",
      "\n",
      "The present study investigates the effect of neutron source on the incineration of Light Water Reactors waste by the Gas Turbine-Modular Helium Reactor. Three different core configurations have been analyzed: critical, accelerator and fusion driven systems. In the accelerator driven system, neutrons are generated by a 1000 MeV proton beam interacting with a lead target. In the fusion driven system, neutrons are generated, in the center of the core, with a fixed energy of 14.1 MeV, through a deuterium-tritium reaction. The comparison of the actinides transmutation for the three configurations shows unclear benefits for the driven systems. \n",
      "\n",
      "------- \n",
      "\n",
      "Quantifying the climate change effects of bioenergy systems : Comparison of 15 impact assessment methods \n",
      "\n",
      "['Brandao, Miguel', 'Kirschbaum, Miko', 'Cowie, Annette', 'Univ New England, NSW', 'Hjuler, Susanne', 'COWI AS, Lyngby'] \n",
      "\n",
      "Ongoing concern over climate change has led to interest in replacing fossil energy with bioenergy. There are different approaches to quantitatively estimate the climate change effects of bioenergy systems. In the present work, we have focused on a range of published impact assessment methods that vary due to conceptual differences in the treatment of biogenic carbon fluxes, the type of climate change impacts they address and differences in time horizon and time preference. Specifically, this paper reviews fifteen different methods and applies these to three hypothetical bioenergy case studies: (a) woody biomass grown on previously forested land; (b) woody biomass grown on previous pasture land; and (b) annual energy crop grown on previously cropped land. Our analysis shows that the choice of method can have an important influence on the quantification of climate change effects of bioenergy, particularly when a mature forest is converted to bioenergy use as it involves a substantial reduction in biomass carbon stocks. Results are more uniform in other case studies. In general, results are more sensitive to specific impact assessment methods when they involve both emissions and removals at different points in time, such as for forest bioenergy, but have a much smaller influence on agricultural bioenergy systems grown on land previously used for pasture or annual cropping. The development of effective policies for climate change mitigation through renewable energy use requires consistent and accurate approaches to identification of bioenergy systems that can result in climate change mitigation. The use of different methods for the same purpose: estimating the climate change effects of bioenergy systems, can lead to confusing and contradictory conclusions. A full interpretation of the results generated with different methods must be based on an understanding that the different methods focus on different aspects of climate change and represent different time preferences. \n",
      "\n",
      "------- \n",
      "\n",
      "How old is streamwater? : Open questions in catchment transit time conceptualization, modelling and analysis \n",
      "\n",
      "['McDonnell, J.J.', 'McGuire, K.', 'Aggarwal, P.', 'Beven, K.', 'Biondi, D.', 'Destouni, G.', 'Dunn, S.', 'James, A.', 'Kirchner, J.', 'Kraft, P.', 'Lyon, S.', 'Maloszewski, P.', 'Newman, B.', 'Pfister, L.', 'Rinaldo, A.', 'Rodhe, A.', 'Sayama, T.', 'Seibert, J.', 'Solomon, K.', 'Soulsby, C.', 'Stewart, M.', 'Tetzlaff, D.', 'Tobin, C.', 'Troch, P.', 'Weiler, M.', 'Western, A.', 'Wörman, Anders', 'Wrede, S.'] \n",
      "\n",
      "The time water spends travelling subsurface through a catchment to the stream network (i.e. the catchment water transit time) fundamentally describes the storage, flow pathway heterogeneity and sources of water in a catchment. The distribution of transit times reflects how catchments retain and release water and solutes that in turn set biogeochemical conditions and affect contamination release or persistence. Thus, quan- tifying the transit time distribution provides an important constraint on biogeochemical processes and catchment sensitivity to anthropogenic inputs, contamination and land-use change. Although the assumptions and limitations of past and present transit time modelling approaches have been recently reviewed (McGuire and McDonnell, 2006), there remain many fundamental research challenges for understanding how transit time can be used to quantify catchment flow processes and aid in the development and testing of rainfall–runoff models. In this Commen- tary study, we summarize what we think are the open research questions in transit time research. These thoughts come from a 3-day workshop in January 2009 at the International Atomic Energy Agency in Vienna. We attempt to lay out a roadmap for this work for the hydrological commu- nity over the next 10 years. We do this by first defining what we mean (qualitatively and quantitatively) by transit time and then organize our vision around needs in transit time theory, needs in field studies of tran- sit time and needs in rainfall – runoff modelling. Our goal in presenting this material is to encourage widespread use of transit time information in process studies to provide new insights to catchment function and to inform the structural development and testing of hydrologic models. \n",
      "\n",
      "------- \n",
      "\n",
      "Microfluidic apoptosis chip for drug screening to improve and personalize cancer therapy \n",
      "\n",
      "['Wolbers, F.', 'Franke, H.', 'Andersson-Svahn, Helene', 'Vermes, I.', 'Van Den Berg, A.'] \n",
      "\n",
      "Currently, the knowledge on breast cancer treatment is increasing, however, there are still hardly any assays present to match patients to the right form of therapy to enhance the therapeutic effectiveness. Therefore a microfluidic device is developed to analyze the hormonal and chemosensitivity of first breast cancer cell lines (MCF-7) and in a later stage a patients' own tumor cells. The apoptotic inducers tumor necrosis factor-α, in combination with cycloheximide, and staurosporine stained MCF-7 cells positive for Annexin V and PI, and showed a decrease in area coverage as compared to untreated cells. \n",
      "\n",
      "------- \n",
      "\n",
      "Towards a deeper understanding of the human brain \n",
      "\n",
      "['Sjöstedt, Evelina'] \n",
      "\n",
      "Identifying the proteome variation in different parts of the body provides fundamental molecular details, enabling further findings and mapping of tissue specific proteins. By combining quantitative transcriptomics with qualitative antibody based proteomics, the Human Protein Atlas (HPA) project aims to protein profile each human protein-coding gene. Genes with varying expression levels in the different tissue types are categorized as tissue elevated in one tissue compared to others, thus connecting genes to potential tissue specific functions. This thesis focuses on the most complex organ in the human body, the brain. With its billions of neurons specifically organized and interconnected, the ability of not only controlling the body but also responsible for higher cognitive functions, the brain is still not fully understood.In my search for brain important proteins, genes were classified at different stages based on expression levels. In Paper I and II the transcriptome of cerebral cortex was compared with peripheral organs to classify genes with elevated expression in the brain. Brain expression information was expanded by including external data (GTEx and FANTOM5) into the analysis, in Paper III. Thereafter, in Paper IV, the three datasets (HPA, GTEx and FANTOM5) were aligned and combined, enabling a consensus classification with an improved representation of the brain complexity. The most recent classification provided whole body gene expression profiles and out of the 19,670 protein-coding genes, 2,501 were expressed at elevated levels in the brain compared to the other tissue types. Twelve individual regions represented the brain as an organ, and were further analyzed and compared for regional classification of gene expression. One thousand genes showed regional variation in expression level, thus classified as regionally elevated within the brain. Interestingly, less than 500 of the genes classified as brain elevated on the whole body level, and were also regionally elevated in the brain. Many genes with regionally variable expression within the brain showed higher expression in a peripheral organ than in the brain when comparing whole body expression. Based on elevated expression in the brain or brain regions, more than 3,000 genes were suggested to be of high importance to the brain.In addition, this high-throughput approach to combine transcriptomics and protein profiles in tissues and cells further generated new knowledge in several different other aspects: better understanding of uncharacterized and “missing proteins” (Paper III), validation of an antibody improving classification of pituitary adenoma (Paper V) and in Paper VI the possibility to explore cancer specific expression in relation to clinical data and normal tissue expression.There are multiple diseases of the brain that are poorly understood on both a cellular and molecular level. While my work mainly focused on identifying and understanding the molecular organization of the normal brain, the ultimate goal of mapping and studying the normal expression baseline is to understand the molecular aspects of disease and identify ways to prevent, treat and cure diseases. \n",
      "\n",
      "------- \n",
      "\n",
      "Genetics meets proteomics : perspectives for large population-based studies \n",
      "\n",
      "['Suhre, K.', 'McCarthy, M.', 'Schwenk, Jochen'] \n",
      "\n",
      "Proteomic analysis of cells, tissues and body fluids has generated valuable insights into the complex processes influencing human biology. Proteins represent intermediate phenotypes for disease and provide insight into how genetic and non-genetic risk factors are mechanistically linked to clinical outcomes. Associations between protein levels and DNA sequence variants that colocalize with risk alleles for common diseases can expose disease-associated pathways, revealing novel drug targets and translational biomarkers. However, genome-wide, population-scale analyses of proteomic data are only now emerging. Here, we review current findings from studies of the plasma proteome and discuss their potential for advancing biomedical translation through the interpretation of genome-wide association analyses. We highlight the challenges faced by currently available technologies and provide perspectives relevant to their future application in large-scale biobank studies. \n",
      "\n",
      "------- \n",
      "\n",
      "Progress on the HUPO Draft Human Proteome : 2017 Metrics of the Human Proteome Project \n",
      "\n",
      "['Omenn, Gilbert', 'Lane, Lydie', 'Lundberg, Emma', 'Oyerall, Christopher', 'Deutsch, Eric'] \n",
      "\n",
      "The Human Proteome Organization (HUPO) Human Proteome Project (HPP) continues to make progress on its two overall goals: (1) completing the protein parts list, with an annual update of the HUPO draft human proteome, and (2) making proteomics an integrated complement to genomics and transcriptomics throughout biomedical and life sciences research. neXtProt version 2017-01-23 has 17 008 confident protein identifications (Protein Existence [PE] level 1) that are compliant with the HPP Guidelines v2.1 (https://hupo.org/Guidelines), up from 13 664 in 2012-12 and 16 518 in 2016-04. Remaining to be found by mass spectrometry and other methods are 2579 \"missing proteins\" (PE2+3+4), down from 2949 in 2016. PeptideAtlas 2017-01 has 15 173 canonical proteins, accounting for nearly all of the 15 290 PE1 proteins based on MS data. These resources have extensive data on PTMs, single amino acid variants, and splice isoforms. The Human Protein Atlas v16 has 10 492 highly curated protein entries with tissue and subcellular spatial localization of proteins and transcript expression. Organ-specific popular protein lists have been generated for broad use in quantitative targeted proteomics using SRM-MS or DIA-SWATH-MS studies of biology and disease. \n",
      "\n",
      "------- \n",
      "\n",
      "Global regulation of yeast metabolism \n",
      "\n",
      "['Nielsen, Jens'] \n",
      "\n",
      "The yeast Saccharomyces cerevisiae is widely used for production of fuels, chemicals, pharmaceuticals andmaterials. Through metabolic engineering of this yeast a number of novel new industrial processes have beendeveloped over the last 10 years. Besides its wide industrial use, S. cerevisiae serves as an eukaryal modelorganism, and many systems biology tools have therefore been developed for this organism. Despite ourextensive knowledge of yeast metabolism and its regulation we are still facing challenges when we want tointegrate this information into mathematical models. In this presentation examples of studies on global regulationof yeast metabolism will be provided. This will include analysis of how yeast rewire its metabolism at the globallevel when exposed to various types of stress and how global regulators like Snf1 and Sir2 controls yeastmetabolism \n",
      "\n",
      "------- \n",
      "\n",
      "Optimizing anaerobic growth rate and fermentation kinetics in Saccharomyces cerevisiae strains expressing Calvin-cycle enzymes for improved ethanol yield \n",
      "\n",
      "['Papapetridis, Ioannis', 'Goudriaan, Maaike', 'Vitali, María', 'Keijzer, Nikita', 'Broek, Marcel', 'van Maris, Antonius', 'Pronk, Jack'] \n",
      "\n",
      "Background: Reduction or elimination of by-product formation is of immediate economic relevance in fermentation processes for industrial bioethanol production with the yeast Saccharomyces cerevisiae. Anaerobic cultures of wildtype S. cerevisiae require formation of glycerol to maintain the intracellular NADH/NAD(+) balance. Previously, functional expression of the Calvin-cycle enzymes ribulose-1,5-bisphosphate carboxylase (RuBisCO) and phosphoribulokinase (PRK) in S. cerevisiae was shown to enable reoxidation of NADH with CO2 as electron acceptor. In slow-growing cultures, this engineering strategy strongly decreased the glycerol yield, while increasing the ethanol yield on sugar. The present study explores engineering strategies to improve rates of growth and alcoholic fermentation in yeast strains that functionally express RuBisCO and PRK, while maximizing the positive impact on the ethanol yield. Results: Multi-copy integration of a bacterial-RuBisCO expression cassette was combined with expression of the Escherichia coli GroEL/GroES chaperones and expression of PRK from the anaerobically inducible DAN1 promoter. In anaerobic, glucose-grown bioreactor batch cultures, the resulting S. cerevisiae strain showed a 31% lower glycerol yield and a 31% lower specific growth rate than a non-engineered reference strain. Growth of the engineered strain in anaerobic, glucose-limited chemostat cultures revealed a negative correlation between its specific growth rate and the contribution of the Calvin-cycle enzymes to redox homeostasis. Additional deletion of GPD2, which encodes an isoenzyme of NAD(+)-dependent glycerol-3-phosphate dehydrogenase, combined with overexpression of the structural genes for enzymes of the non-oxidative pentose-phosphate pathway, yielded a CO2-reducing strain that grew at the same rate as a non-engineered reference strain in anaerobic bioreactor batch cultures, while exhibiting a 86% lower glycerol yield and a 15% higher ethanol yield. Conclusions: The metabolic engineering strategy presented here enables an almost complete elimination of glycerol production in anaerobic, glucose-grown batch cultures of S. cerevisiae, with an associated increase in ethanol yield, while retaining near wild-type growth rates and a capacity for glycerol formation under osmotic stress. Using current genome-editing techniques, the required genetic modifications can be introduced in one or a few transformations. Evaluation of this concept in industrial strains and conditions is therefore a realistic next step towards its implementation for improving the efficiency of first-and second-generation bioethanol production. \n",
      "\n",
      "------- \n",
      "\n",
      "Deltaproteobacteria and Spirochaetes-Like Bacteria Are Abundant Putative Mercury Methylators in Oxygen-Deficient Water and Marine Particles in the Baltic Sea \n",
      "\n",
      "['Capo, Eric', 'Swedish Univ Agr Sci, Dept', 'Bravo, Andrea', 'Soerensen, Anne', 'Bertilsson, Stefan', 'Pinhassi, Jarone', 'Feng, Caiyan', 'Andersson, Anders', 'Buck, Moritz', 'Björn, Erik'] \n",
      "\n",
      "Methylmercury (MeHg), a neurotoxic compound biomagnifying in aquatic food webs, can be a threat to human health via fish consumption. However, the composition and distribution of the microbial communities mediating the methylation of mercury (Hg) to MeHg in marine systems remain largely unknown. In order to fill this knowledge gap, we used the Baltic Sea Reference Metagenome (BARM) dataset to study the abundance and distribution of the genes involved in Hg methylation (thehgcABgene cluster). We determined the relative abundance of thehgcABgenes and their taxonomic identity in 81 brackish metagenomes that cover spatial, seasonal and redox variability in the Baltic Sea water column. ThehgcABgenes were predominantly detected in anoxic water, but somehgcABgenes were also detected in hypoxic and normoxic waters. Phylogenetic analysis identified putative Hg methylators within Deltaproteobacteria, in oxygen-deficient water layers, but also Spirochaetes-like and Kiritimatiellaeota-like bacteria. Higher relative quantities ofhgcABgenes were found in metagenomes from marine particles compared to free-living communities in anoxic water, suggesting that such particles are hotspot habitats for Hg methylators in oxygen-depleted seawater. Altogether, our work unveils the diversity of the microorganisms with the potential to mediate MeHg production in the Baltic Sea and pinpoint the important ecological niches for these microorganisms within the marine water column. \n",
      "\n",
      "------- \n",
      "\n",
      "Validation of trace code against ROSA/LSTF test for SBLOCA of pressure vessel upper-head small break \n",
      "\n",
      "['Xing, M.', 'Hu, X.', 'Chen, Y.', 'Li, L.', 'Ma, Weimin'] \n",
      "\n",
      "OECD/NEA ROSA/LSTF project tests are performed on the Large Scale Test Facility (LSTF). LSTF is a full-height, fullpressure and 1/48 volumetrically-scaled two-loop system which aims to simulate Japanese Tsuruga-2 Westinghouse-type 4-loop PWR. ROSA-V Test 6-1 simulates a pressure vessel (PV) upper-head small break loss-of-coolant accident (SBLOCA) with a break size equivalent to 1.9% of the volumetrically scaled cross-sectional area of the reference PWR cold leg. By building a TRACE calculation model of LSTF and PV upper-head, the paper dedicated to assess the effect of different modeling options and parameters on simulating thermal hydraulic behaviors of TRACE code. The results show that TRACE code well reproduce the physical phenomena involved in this type of SBLOCA scenarios. Almost all the events in the experiment are well predicted by the model based on TRACE code. In addition, the sensitivity of different models and parameters are investigated. For example, the code slightly overestimated the break mass flow from upper head which could affect the accuracy of the results significantly. The rising of core exit temperature (CET) is significantly influenced by the bypass flow area between downcomer and hot leg. \n",
      "\n",
      "------- \n",
      "\n",
      "Publication Committee Meeting HUPO 5th Annual World Congress Long Beach, CA, USA 30 October 2006 \n",
      "\n",
      "['Orchard, S.', 'Heck, A.', 'Uhlén, Mathias', 'Ping, P.'] \n",
      "\n",
      "This meeting brought together delegates from industry, academia and the publishing houses to facilitate discussions on the level of support from the journals for the use of standardised data formats and their interest in the creation of a network of proteomics repositories collaborating on a coordinated data curation effort. Discussions centred on how best to structure interactions between journals, databases and researchers to improve accessibility to data, and facilitate comparisons between datasets. \n",
      "\n",
      "------- \n",
      "\n",
      "Experimental insights into the importance of aquatic bacterial community composition to the degradation of dissolved organic matter \n",
      "\n",
      "['Logue, Jurg', 'Stedmon, Colin', 'Kellerman, Anne', 'Nielsen, Nikoline', 'Andersson, Anders', 'Laudon, Hjalmar', 'Lindstrom, Eva', 'Kritzberg, Emma'] \n",
      "\n",
      "Bacteria play a central role in the cycling of carbon, yet our understanding of the relationship between the taxonomic composition and the degradation of dissolved organic matter (DOM) is still poor. In this experimental study, we were able to demonstrate a direct link between community composition and ecosystem functioning in that differently structured aquatic bacterial communities differed in their degradation of terrestrially derived DOM. Although the same amount of carbon was processed, both the temporal pattern of degradation and the compounds degraded differed among communities. We, moreover, uncovered that low-molecular-weight carbon was available to all communities for utilisation, whereas the ability to degrade carbon of greater molecular weight was a trait less widely distributed. Finally, whereas the degradation of either low-or high-molecular-weight carbon was not restricted to a single phylogenetic clade, our results illustrate that bacterial taxa of similar phylogenetic classification differed substantially in their association with the degradation of DOM compounds. Applying techniques that capture the diversity and complexity of both bacterial communities and DOM, our study provides new insight into how the structure of bacterial communities may affect processes of biogeochemical significance. \n",
      "\n",
      "------- \n",
      "\n"
     ]
    }
   ],
   "source": [
    "\n",
    "for i, row in X.iterrows():\n",
    "    print(row.Title, \"\\n\")\n",
    "    print(row.AllName.rstrip(\"[\").lstrip(\"']\"),\"\\n\")\n",
    "    print(row.Abstract, \"\\n\")\n",
    "    print(\"------- \\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d60c07ef",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "51b45e3b",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b3a8fd6e",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "9dab5e56",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 96/96 [00:09<00:00, 10.34it/s]\n"
     ]
    }
   ],
   "source": [
    "model = model.cuda()\n",
    "model.eval()\n",
    "dataloader = torch.utils.data.DataLoader(test_dataset, batch_size=256)\n",
    "acc_list = list()\n",
    "with torch.no_grad():\n",
    "    for b in tqdm(dataloader):\n",
    "        b = {k:v.cuda() for k, v in b.items()}\n",
    "        output = model(**b)\n",
    "        acc = sum([p==v for p,v in zip(torch.argmax(output['logits_per_image'],0), np.arange(256))]) / 256\n",
    "        acc_list.append(acc.cpu().numpy()[None])\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "181e12b6",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python (recommomendationRanker)",
   "language": "python",
   "name": "recommomendationranker"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
